******* 001_Two_Sum ***********

FAILED 001_Two_Sum/test_001_Two_Sum.py::test_twoSum - assert [0, 1] == [0, 5]
============================== 1 failed in 0.01s ===============================

******* 002_Add_Two_Numbers ***********

FAILED 002_Add_Two_Numbers/test_002_Add_Two_Numbers.py::TestAddTwoNumbers::test_base_case
FAILED 002_Add_Two_Numbers/test_002_Add_Two_Numbers.py::TestAddTwoNumbers::test_zeros
FAILED 002_Add_Two_Numbers/test_002_Add_Two_Numbers.py::TestAddTwoNumbers::test_different_length
FAILED 002_Add_Two_Numbers/test_002_Add_Two_Numbers.py::TestAddTwoNumbers::test_carry_over
FAILED 002_Add_Two_Numbers/test_002_Add_Two_Numbers.py::TestAddTwoNumbers::test_empty_lists
============================== 5 failed in 0.03s ===============================

******* 003_Longest_Substring_Without_Repeating_Characters ***********

============================== 7 passed in 0.01s ===============================

******* 004_Median_of_Two_Sorted_Arrays ***********

============================== 1 passed in 0.00s ===============================

******* 005_Longest_Palindromic_Substring ***********

========================= 4 failed, 3 passed in 0.02s ==========================
FAILED 005_Longest_Palindromic_Substring/test_005_Longest_Palindromic_Substring.py::test_longest_palindrome_odd_length_string
FAILED 005_Longest_Palindromic_Substring/test_005_Longest_Palindromic_Substring.py::test_longest_palindrome_even_length_string
FAILED 005_Longest_Palindromic_Substring/test_005_Longest_Palindromic_Substring.py::test_longest_palindrome_multiple_palindromes
FAILED 005_Longest_Palindromic_Substring/test_005_Longest_Palindromic_Substring.py::test_longest_palindrome_full_string_palindrome
========================= 4 failed, 3 passed in 0.02s ==========================

******* 006_ZigZag_Conversion ***********

============================== 1 passed in 0.00s ===============================

******* 007_Reverse_Integer ***********

============================== 1 passed in 0.00s ===============================

******* 008_String_to_Integer(atoi) ***********

============================== 6 passed in 0.01s ===============================

******* 009_Palindrome_Number ***********

========================= 1 failed, 2 passed in 0.01s ==========================
E       Failed: DID NOT RAISE <class 'TypeError'>
009_Palindrome_Number/test_009_Palindrome_Number.py:22: Failed
FAILED 009_Palindrome_Number/test_009_Palindrome_Number.py::test_isPalindrome_error
========================= 1 failed, 2 passed in 0.01s ==========================

******* 010_Regular_Expression_Matching ***********


******* 011_Container_With_Most_Water ***********

    from solution import Solution

******* 012_Integer_to_Roman ***********

============================== 1 passed in 0.00s ===============================

******* 013_Roman_to_Integer ***********

FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_I
FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_V
FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_IV
FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_IX
FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_XL
FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_CMIX
FAILED 013_Roman_to_Integer/test_013_Roman_to_Integer.py::TestSolution::test_romanToInt_MMMCMXCIX
============================== 7 failed in 0.02s ===============================

******* 014_Longest_Common_Prefix ***********

============================== 5 passed in 0.01s ===============================

******* 015_3Sum ***********


******* 016_3Sum_Closest ***********

FAILED 016_3Sum_Closest/test_016_3Sum_Closest.py::test_threeSumClosest - Inde...
============================== 1 failed in 0.01s ===============================

******* 017_Letter_Combinations_of_a_Phone_Number ***********


******* 018_4Sum ***********

FAILED 018_4Sum/test_018_4Sum.py::test_four_sum - assert [(0, 0, 0, 0)] == [(...
============================== 1 failed in 0.01s ===============================

******* 019_Remove_Nth_Node_From_End_of_List ***********

FAILED 019_Remove_Nth_Node_From_End_of_List/test_019_Remove_Nth_Node_From_End_of_List.py::test_removeNthFromEnd_empty_list
FAILED 019_Remove_Nth_Node_From_End_of_List/test_019_Remove_Nth_Node_From_End_of_List.py::test_removeNthFromEnd_invalid_n
FAILED 019_Remove_Nth_Node_From_End_of_List/test_019_Remove_Nth_Node_From_End_of_List.py::test_removeNthFromEnd_remove_head
FAILED 019_Remove_Nth_Node_From_End_of_List/test_019_Remove_Nth_Node_From_End_of_List.py::test_removeNthFromEnd_remove_tail
FAILED 019_Remove_Nth_Node_From_End_of_List/test_019_Remove_Nth_Node_From_End_of_List.py::test_removeNthFromEnd_remove_middle
============================== 5 failed in 0.02s ===============================

******* 020_Valid_Parentheses ***********

============================== 1 passed in 0.00s ===============================

******* 021_Merge_Two_Sorted_Lists ***********

FAILED 021_Merge_Two_Sorted_Lists/test_021_Merge_Two_Sorted_Lists.py::TestMergeTwoLists::test_mergeTwoLists
============================== 1 failed in 0.01s ===============================

******* 022_Generate_Parentheses ***********

FAILED 022_Generate_Parentheses/test_022_Generate_Parentheses.py::test_generateParenthesis
============================== 1 failed in 0.01s ===============================

******* 023_Merge_k_Sorted_Lists ***********

========================= 3 failed, 1 passed in 0.03s ==========================
FAILED 023_Merge_k_Sorted_Lists/test_023_Merge_k_Sorted_Lists.py::testMergeKListsPriorityQueue
FAILED 023_Merge_k_Sorted_Lists/test_023_Merge_k_Sorted_Lists.py::testMergeKListsSort
FAILED 023_Merge_k_Sorted_Lists/test_023_Merge_k_Sorted_Lists.py::testMergeKListsRecursive
========================= 3 failed, 1 passed in 0.03s ==========================

******* 024_Swap_Nodes_in_Pairs ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/024_Swap_Nodes_in_Pairs/test_024_Swap_Nodes_in_Pairs.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution, ListNode
E   ImportError: attempted relative import with no known parent package

******* 025_Reverse_Nodes_i_ ***********


******* 025_Reverse_Nodes_i_ k-Group ***********

FAILED 025_Reverse_Nodes_i_ k-Group/test_025_Reverse_Nodes_i_ k-Group.py::TestSolution::test_reverseKGroup
============================== 1 failed in 0.02s ===============================

******* 026_Remove_Duplicates_from_Sorted_Array ***********

============================== 10 passed in 0.01s ==============================

******* 027_Remove_Element ***********

FAILED 027_Remove_Element/test_027_Remove_Element.py::test_empty_list - NameE...
FAILED 027_Remove_Element/test_027_Remove_Element.py::test_remove_all_elements
FAILED 027_Remove_Element/test_027_Remove_Element.py::test_remove_no_elements
FAILED 027_Remove_Element/test_027_Remove_Element.py::test_remove_some_elements
FAILED 027_Remove_Element/test_027_Remove_Element.py::test_remove_just_one_element
FAILED 027_Remove_Element/test_027_Remove_Element.py::test_remove_with_multiple_occurrences
============================== 6 failed in 0.02s ===============================

******* 028_Implement_strStr() ***********

FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_strStr_empty_needle
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_strStr_no_match
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_strStr_single_char_haystack
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_strStr_multiple_matches
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_strStr_match_at_end
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_makeNext
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_makeNext_empty_needle
FAILED 028_Implement_strStr()/test_028_Implement_strStr().py::TestSolution::test_makeNext_single_char_needle
============================== 8 failed in 0.02s ===============================

******* 029_Divide_Two_Integers ***********

========================= 4 failed, 3 passed in 0.02s ==========================
FAILED 029_Divide_Two_Integers/test_029_Divide_Two_Integers.py::test_divide_by_zero
FAILED 029_Divide_Two_Integers/test_029_Divide_Two_Integers.py::test_positive_dividend_and_divisor
FAILED 029_Divide_Two_Integers/test_029_Divide_Two_Integers.py::test_negative_dividend_and_divisor
FAILED 029_Divide_Two_Integers/test_029_Divide_Two_Integers.py::test_negative_dividend_and_positive_divisor
========================= 4 failed, 3 passed in 0.02s ==========================

******* 030_Substring_with_Concatenation_of_All_Words ***********

============================== 1 passed in 0.00s ===============================

******* 031_Next_Permutation ***********

FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_nextPermutation_example1
FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_nextPermutation_example2
FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_nextPermutation_example3
FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_nextPermutation_singleElement
FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_nextPermutation_emptyList
FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_swap_sameIndex
FAILED 031_Next_Permutation/test_031_Next_Permutation.py::TestSolution::test_swap_differentIndex
============================== 7 failed in 0.02s ===============================

******* 032_Longest_Valid_Parentheses ***********


******* 033_Search_in_Rotated_Sorted_Array ***********

FAILED 033_Search_in_Rotated_Sorted_Array/test_033_Search_in_Rotated_Sorted_Array.py::test_search
============================== 1 failed in 0.01s ===============================

******* 034_Search_for_a_Range ***********

FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_emptyList
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_notInList
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_oneOccurrence
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_multipleOccurrences
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_firstElement
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_lastElement
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_singleElement
FAILED 034_Search_for_a_Range/test_034_Search_for_a_Range.py::TestSolution::test_searchRange_negativeNumbers
============================== 8 failed in 0.02s ===============================

******* 035_Search_Insert_Position ***********

FAILED 035_Search_Insert_Position/test_035_Search_Insert_Position.py::test_searchInsert
============================== 1 failed in 0.01s ===============================

******* 036_Valid ***********


******* 036_Valid Sudoku ***********

========================= 2 failed, 1 passed in 0.02s ==========================
FAILED 036_Valid Sudoku/test_036_Valid Sudoku.py::test_valid_sudoku - TypeErr...
FAILED 036_Valid Sudoku/test_036_Valid Sudoku.py::test_invalid_sudoku - TypeE...
========================= 2 failed, 1 passed in 0.02s ==========================

******* 037_Sudoku_Solver ***********

FAILED 037_Sudoku_Solver/test_037_Sudoku_Solver.py::test_solveSudoku - TypeEr...
============================== 1 failed in 0.02s ===============================

******* 038_Count_and_Say ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/038_Count_and_Say/test_038_Count_and_Say.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution
E   ImportError: attempted relative import with no known parent package

******* 039_Combination_Sum ***********

FAILED 039_Combination_Sum/test_039_Combination_Sum.py::TestSolution::test_combinationSum
============================== 1 failed in 0.01s ===============================

******* 040_Combination_Sum_II ***********

========================= 3 failed, 2 passed in 0.02s ==========================
FAILED 040_Combination_Sum_II/test_040_Combination_Sum_II.py::TestSolution::test_combinationSum2_1
FAILED 040_Combination_Sum_II/test_040_Combination_Sum_II.py::TestSolution::test_combinationSum2_4
FAILED 040_Combination_Sum_II/test_040_Combination_Sum_II.py::TestSolution::test_combinationSum2_5
========================= 3 failed, 2 passed in 0.02s ==========================

******* 041_First_Missing_Positive ***********

FAILED 041_First_Missing_Positive/test_041_First_Missing_Positive.py::TestSolution::test_firstMissingPositive
FAILED 041_First_Missing_Positive/test_041_First_Missing_Positive.py::TestSolution::test_firstMissingPositive_invalid_input
============================== 2 failed in 0.01s ===============================

******* 042_Trapping_Rain_Water ***********

========================= 1 failed, 1 passed in 0.01s ==========================
FAILED 042_Trapping_Rain_Water/test_042_Trapping_Rain_Water.py::test_rain_water
========================= 1 failed, 1 passed in 0.01s ==========================

******* 043_Multiply_Strings ***********

FAILED 043_Multiply_Strings/test_043_Multiply_Strings.py::test_multiply - Ass...
============================== 1 failed in 0.01s ===============================

******* 044_Wildcard_Matching ***********

FAILED 044_Wildcard_Matching/test_044_Wildcard_Matching.py::test_isMatch1 - N...
FAILED 044_Wildcard_Matching/test_044_Wildcard_Matching.py::test_isMatch2 - N...
FAILED 044_Wildcard_Matching/test_044_Wildcard_Matching.py::test_isMatch3 - N...
FAILED 044_Wildcard_Matching/test_044_Wildcard_Matching.py::test_isMatch4 - N...
FAILED 044_Wildcard_Matching/test_044_Wildcard_Matching.py::test_isMatch5 - N...
============================== 5 failed in 0.02s ===============================

******* 045_Jump_Game_II ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/045_Jump_Game_II/test_045_Jump_Game_II.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from ...path.to import Solution
E   ImportError: attempted relative import with no known parent package

******* 046_Permutations ***********

FAILED 046_Permutations/test_046_Permutations.py::test_permute_using_itertools
FAILED 046_Permutations/test_046_Permutations.py::test_permute_using_DPS_with_swapping
FAILED 046_Permutations/test_046_Permutations.py::test_permute_iterative_solution
============================== 3 failed in 0.01s ===============================

******* 047_Permutations_II ***********

FAILED 047_Permutations_II/test_047_Permutations_II.py::test_permuteUnique_1
FAILED 047_Permutations_II/test_047_Permutations_II.py::test_permuteUnique_2
FAILED 047_Permutations_II/test_047_Permutations_II.py::test_permuteUnique_3
============================== 3 failed in 0.02s ===============================

******* 048_Rotate_Image ***********

========================= 3 failed, 2 passed in 0.02s ==========================
FAILED 048_Rotate_Image/test_048_Rotate_Image.py::test_rotate_4x4_matrix - Ty...
FAILED 048_Rotate_Image/test_048_Rotate_Image.py::test_rotate_empty_matrix - ...
FAILED 048_Rotate_Image/test_048_Rotate_Image.py::test_rotate_odd_matrix - Ty...
========================= 3 failed, 2 passed in 0.02s ==========================

******* 049_Group_Anagrams ***********

============================== 1 passed in 0.00s ===============================

******* 050_Pow(x, ***********


******* 050_Pow(x, n) ***********

============================== 1 passed in 0.00s ===============================

******* 051_N-Queens ***********


******* 052_N-Queens ***********


******* 052_N-Queens II ***********

FAILED 052_N-Queens II/test_052_N-Queens II.py::test_totalNQueens - assert 1 ...
============================== 1 failed in 0.01s ===============================

******* 053_Maximum_Subarray ***********

FAILED 053_Maximum_Subarray/test_053_Maximum_Subarray.py::TestSolution::test_maxSubArray
============================== 1 failed in 0.01s ===============================

******* 054_Spiral_Matrix ***********

========================= 1 failed, 2 passed in 0.02s ==========================
FAILED 054_Spiral_Matrix/test_054_Spiral_Matrix.py::test_get_spiralOrder - as...
========================= 1 failed, 2 passed in 0.02s ==========================

******* 055_Jump_Game ***********


******* 056_Merge_Intervals ***********

FAILED 056_Merge_Intervals/test_056_Merge_Intervals.py::test_merge_invalid_input
FAILED 056_Merge_Intervals/test_056_Merge_Intervals.py::test_merge_one_element
FAILED 056_Merge_Intervals/test_056_Merge_Intervals.py::test_merge_two_non_overlapping_intervals
FAILED 056_Merge_Intervals/test_056_Merge_Intervals.py::test_merge_two_overlapping_intervals
FAILED 056_Merge_Intervals/test_056_Merge_Intervals.py::test_merge_multiple_overlapping_intervals
============================== 5 failed in 0.02s ===============================

******* 057_Insert_Interval ***********


******* 058_Length_of_Last_Word ***********

========================= 1 failed, 9 passed in 0.02s ==========================
FAILED 058_Length_of_Last_Word/test_058_Length_of_Last_Word.py::test_numbers_in_string
========================= 1 failed, 9 passed in 0.02s ==========================

******* 059_Spiral_Matrix_II ***********

FAILED 059_Spiral_Matrix_II/test_059_Spiral_Matrix_II.py::TestSolution::test_generateMatrix
============================== 1 failed in 0.01s ===============================

******* 060_Permutation_Sequence ***********

========================= 3 failed, 1 passed in 0.02s ==========================
FAILED 060_Permutation_Sequence/test_060_Permutation_Sequence.py::test_getPermutation[3-2-132]
FAILED 060_Permutation_Sequence/test_060_Permutation_Sequence.py::test_getPermutation[4-9-2314]
FAILED 060_Permutation_Sequence/test_060_Permutation_Sequence.py::test_getPermutation_negative_cases
========================= 3 failed, 1 passed in 0.02s ==========================

******* 061_Rotate_List ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/061_Rotate_List/test_061_Rotate_List.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, ListNode
E   ImportError: cannot import name 'ListNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/061_Rotate_List/solution.py)

******* 062_Unique_Paths ***********

========================= 2 failed, 1 passed in 0.02s ==========================
FAILED 062_Unique_Paths/test_062_Unique_Paths.py::test_uniquePaths - IndexErr...
FAILED 062_Unique_Paths/test_062_Unique_Paths.py::test_uniquePaths_raises_ValueError
========================= 2 failed, 1 passed in 0.02s ==========================

******* 063_Unique_Paths_II ***********

FAILED 063_Unique_Paths_II/test_063_Unique_Paths_II.py::test_uniquePathsWithObstacles
============================== 1 failed in 0.01s ===============================

******* 064_Minimum_Path_Sum ***********

FAILED 064_Minimum_Path_Sum/test_064_Minimum_Path_Sum.py::test_minPathSum - I...
============================== 1 failed in 0.01s ===============================

******* 065_Valid_Number ***********

========================= 1 failed, 14 passed in 0.02s =========================
FAILED 065_Valid_Number/test_065_Valid_Number.py::test_mixed_notation - Asser...
========================= 1 failed, 14 passed in 0.02s =========================

******* 066_Plus_One ***********

========================= 1 failed, 1 passed in 0.01s ==========================
FAILED 066_Plus_One/test_066_Plus_One.py::test_plusOne - assert [0, 0, 1] == ...
========================= 1 failed, 1 passed in 0.01s ==========================

******* 067_Add_Binary ***********

============================== 1 passed in 0.00s ===============================

******* 068_Text_Justification ***********

FAILED 068_Text_Justification/test_068_Text_Justification.py::TestSolution::test_fullJustify
============================== 1 failed in 0.01s ===============================

******* 069_Sqrt(x) ***********

FAILED 069_Sqrt(x)/test_069_Sqrt(x).py::test_mySqrt_zero - NameError: name 'S...
FAILED 069_Sqrt(x)/test_069_Sqrt(x).py::test_mySqrt_one - NameError: name 'So...
FAILED 069_Sqrt(x)/test_069_Sqrt(x).py::test_mySqrt_four - NameError: name 'S...
FAILED 069_Sqrt(x)/test_069_Sqrt(x).py::test_mySqrt_nine - NameError: name 'S...
FAILED 069_Sqrt(x)/test_069_Sqrt(x).py::test_mySqrt_twenty_five - NameError: ...
FAILED 069_Sqrt(x)/test_069_Sqrt(x).py::test_mySqrt_negative - NameError: nam...
============================== 6 failed in 0.02s ===============================

******* 070_Climbing_Stairs ***********

============================== 2 passed in 0.00s ===============================

******* 071_Simplify_Path ***********


******* 072_Edit_Distance ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked
    from solution import Solution

******* 073_Set_Matrix_Zeroes ***********

FAILED 073_Set_Matrix_Zeroes/test_073_Set_Matrix_Zeroes.py::test_solution - a...
============================== 1 failed in 0.01s ===============================

******* 074_Search_a_2D_Matrix ***********

FAILED 074_Search_a_2D_Matrix/test_074_Search_a_2D_Matrix.py::TestSolution::test_searchMatrix_with_empty_matrix
FAILED 074_Search_a_2D_Matrix/test_074_Search_a_2D_Matrix.py::TestSolution::test_searchMatrix_with_invalid_matrix_values
FAILED 074_Search_a_2D_Matrix/test_074_Search_a_2D_Matrix.py::TestSolution::test_searchMatrix_with_matrix_values_in_range
FAILED 074_Search_a_2D_Matrix/test_074_Search_a_2D_Matrix.py::TestSolution::test_searchMatrix_with_matrix_values_out_of_range
FAILED 074_Search_a_2D_Matrix/test_074_Search_a_2D_Matrix.py::TestSolution::test_searchMatrix_with_same_values
FAILED 074_Search_a_2D_Matrix/test_074_Search_a_2D_Matrix.py::TestSolution::test_searchMatrix_with_negative_values
============================== 6 failed in 0.02s ===============================

******* 075_Sort_Colors ***********

FAILED 075_Sort_Colors/test_075_Sort_Colors.py::test_sortColors_1 - NameError...
FAILED 075_Sort_Colors/test_075_Sort_Colors.py::test_sortColors_2 - NameError...
FAILED 075_Sort_Colors/test_075_Sort_Colors.py::test_sortColors_3 - NameError...
============================== 3 failed in 0.01s ===============================

******* 076_Minimum_Window_Substring ***********

FAILED 076_Minimum_Window_Substring/test_076_Minimum_Window_Substring.py::test_minWindow
FAILED 076_Minimum_Window_Substring/test_076_Minimum_Window_Substring.py::test_first_match
============================== 2 failed in 0.02s ===============================

******* 077_Combinations ***********

FAILED 077_Combinations/test_077_Combinations.py::test_get_combine - NameErro...
FAILED 077_Combinations/test_077_Combinations.py::test_combine - NameError: n...
============================== 2 failed in 0.01s ===============================

******* 078_Subsets ***********

FAILED 078_Subsets/test_078_Subsets.py::TestSolution::test_subsets - assert [...
============================== 1 failed in 0.02s ===============================

******* 079_Word_Search ***********

E       ========================== 6 passed in 0.01s ==========================
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked

******* 080_Remove_Duplicates_from_Sorted_Array_II ***********

============================== 1 passed in 0.00s ===============================

******* 081_Search_in_Rotated_Sorted_Array_II ***********

FAILED 081_Search_in_Rotated_Sorted_Array_II/test_081_Search_in_Rotated_Sorted_Array_II.py::test_search[nums0-0-True]
FAILED 081_Search_in_Rotated_Sorted_Array_II/test_081_Search_in_Rotated_Sorted_Array_II.py::test_search[nums1-3-False]
FAILED 081_Search_in_Rotated_Sorted_Array_II/test_081_Search_in_Rotated_Sorted_Array_II.py::test_search[nums2-0-False]
FAILED 081_Search_in_Rotated_Sorted_Array_II/test_081_Search_in_Rotated_Sorted_Array_II.py::test_search[nums3-1-True]
FAILED 081_Search_in_Rotated_Sorted_Array_II/test_081_Search_in_Rotated_Sorted_Array_II.py::test_search[nums4-3-True]
============================== 5 failed in 0.02s ===============================

******* 083_Remove_Duplicates_from_Sorted_List ***********


******* 084_Largest_Rectangle_in_Histogram ***********


******* 085_Maximal_Rectangle ***********

FAILED 085_Maximal_Rectangle/test_085_Maximal_Rectangle.py::test_maximalRectangle
============================== 1 failed in 0.01s ===============================

******* 086_Partition_List ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/086_Partition_List/test_086_Partition_List.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, ListNode
E   ImportError: cannot import name 'ListNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/086_Partition_List/solution.py)

******* 087_Scramble_String ***********

FAILED 087_Scramble_String/test_087_Scramble_String.py::TestSolution::test_isScramble_basic
FAILED 087_Scramble_String/test_087_Scramble_String.py::TestSolution::test_isScramble_edge_cases
FAILED 087_Scramble_String/test_087_Scramble_String.py::TestSolution::test_isScramble_memoization
FAILED 087_Scramble_String/test_087_Scramble_String.py::TestSolution::test_isScramble_invalid_inputs
============================== 4 failed in 0.02s ===============================

******* 088_Merge_Sorted_Array ***********


******* 089_Gray_Code ***********

========================= 2 failed, 1 passed in 0.02s ==========================
E       Failed: DID NOT RAISE <class 'ValueError'>
089_Gray_Code/test_089_Gray_Code.py:17: Failed
FAILED 089_Gray_Code/test_089_Gray_Code.py::test_grayCode_empty - assert not [0]
FAILED 089_Gray_Code/test_089_Gray_Code.py::test_grayCode_invalid_input - Fai...
========================= 2 failed, 1 passed in 0.02s ==========================

******* 090_Subsets_II ***********

FAILED 090_Subsets_II/test_090_Subsets_II.py::TestSolution::test_subsetsWithDup
============================== 1 failed in 0.01s ===============================

******* 091_Decode_Ways ***********

============================== 1 passed in 0.00s ===============================

******* 092_Reverse_Linked_Lis_ ***********


******* 092_Reverse_Linked_Lis_ II ***********

FAILED 092_Reverse_Linked_Lis_ II/test_092_Reverse_Linked_Lis_ II.py::TestSolution::test_reverseBetween
============================== 1 failed in 0.01s ===============================

******* 093_Restore_IP_Addresses ***********

FAILED 093_Restore_IP_Addresses/test_093_Restore_IP_Addresses.py::test_restoreIpAddresses
FAILED 093_Restore_IP_Addresses/test_093_Restore_IP_Addresses.py::test_isValid
FAILED 093_Restore_IP_Addresses/test_093_Restore_IP_Addresses.py::test_getremainIP
FAILED 093_Restore_IP_Addresses/test_093_Restore_IP_Addresses.py::test_edge_cases
============================== 4 failed in 0.02s ===============================

******* 094_Binary_Tree_Inorder_Traversal ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/094_Binary_Tree_Inorder_Traversal/test_094_Binary_Tree_Inorder_Traversal.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, TreeNode
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/094_Binary_Tree_Inorder_Traversal/solution.py)

******* 095_Unique_Binary_Search_Trees_II ***********

========================= 2 failed, 1 passed in 0.01s ==========================
FAILED 095_Unique_Binary_Search_Trees_II/test_095_Unique_Binary_Search_Trees_II.py::TestSolution::test_single_node
FAILED 095_Unique_Binary_Search_Trees_II/test_095_Unique_Binary_Search_Trees_II.py::TestSolution::test_multiple_nodes
========================= 2 failed, 1 passed in 0.01s ==========================

******* 096_Unique_Binary_Search_Trees ***********

FAILED 096_Unique_Binary_Search_Trees/test_096_Unique_Binary_Search_Trees.py::test_numTrees
============================== 1 failed in 0.01s ===============================

******* 097_Interleaving_String ***********

============================== 1 passed in 0.00s ===============================

******* 098_Validate_Binary_Search_Tree ***********

========================= 1 failed, 1 passed in 0.01s ==========================
FAILED 098_Validate_Binary_Search_Tree/test_098_Validate_Binary_Search_Tree.py::test_isVaild_helper
========================= 1 failed, 1 passed in 0.01s ==========================

******* 099_Recover_Binary_Search_Tree ***********

FAILED 099_Recover_Binary_Search_Tree/test_099_Recover_Binary_Search_Tree.py::test_recoverTree
============================== 1 failed in 0.01s ===============================

******* 100_Same_Tree ***********

============================== 1 passed in 0.00s ===============================

******* 101_Symmetric_Tree ***********

FAILED 101_Symmetric_Tree/test_101_Symmetric_Tree.py::test_isSymmetric - Name...
============================== 1 failed in 0.01s ===============================

******* 102_Binary_Tree_Level_Order_Traversal ***********

============================== 4 passed in 0.01s ===============================

******* 103_Binary_Tree_Zigzag_Level_Order_Traversal ***********

============================== 4 passed in 0.01s ===============================

******* 104_Maximum_Depth_of_Binary_Tree ***********

============================== 2 passed in 0.00s ===============================

******* 105_Construct_Binary_Tree_from_Preorder_and_Inorder_Traversal ***********


******* 1064_Fixed_Point ***********

========================= 2 failed, 2 passed in 0.01s ==========================
FAILED 1064_Fixed_Point/test_1064_Fixed_Point.py::test_fixedPoint - assert 4 ...
FAILED 1064_Fixed_Point/test_1064_Fixed_Point.py::test_fixedPoint_duplicates
========================= 2 failed, 2 passed in 0.01s ==========================

******* 106_Construct_Binary_Tree_from_Inorder_and_Postorder_Traversal ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/106_Construct_Binary_Tree_from_Inorder_and_Postorder_Traversal/test_106_Construct_Binary_Tree_from_Inorder_and_Postorder_Traversal.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import TreeNode, Solution
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/106_Construct_Binary_Tree_from_Inorder_and_Postorder_Traversal/solution.py)

******* 107_Binary_Tree_Level_Order_Traversal_II ***********

========================= 1 failed, 4 passed in 0.02s ==========================
FAILED 107_Binary_Tree_Level_Order_Traversal_II/test_107_Binary_Tree_Level_Order_Traversal_II.py::test_level_order_bottom_unbalanced_tree
========================= 1 failed, 4 passed in 0.02s ==========================

******* 1089_Duplicate_Zeros ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/1089_Duplicate_Zeros/test_1089_Duplicate_Zeros.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution
E   ImportError: attempted relative import with no known parent package

******* 108_Convert_Sorted_Array_to_Binary_Search_Tree ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/108_Convert_Sorted_Array_to_Binary_Search_Tree/test_108_Convert_Sorted_Array_to_Binary_Search_Tree.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, TreeNode
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/108_Convert_Sorted_Array_to_Binary_Search_Tree/solution.py)

******* 109_Convert_Sorted_List_to_Binary_Search_Tree ***********

FAILED 109_Convert_Sorted_List_to_Binary_Search_Tree/test_109_Convert_Sorted_List_to_Binary_Search_Tree.py::test_Solution
============================== 1 failed in 0.01s ===============================

******* 1108_Defanging_an_IP_Address ***********

============================== 3 passed in 0.01s ===============================

******* 110_Balanced_Binary_Tree ***********

========================= 6 failed, 3 passed in 0.02s ==========================
FAILED 110_Balanced_Binary_Tree/test_110_Balanced_Binary_Tree.py::test_solution_balanced_tree
FAILED 110_Balanced_Binary_Tree/test_110_Balanced_Binary_Tree.py::test_solution_unbalanced_tree
FAILED 110_Balanced_Binary_Tree/test_110_Balanced_Binary_Tree.py::test_solution_empty_tree
FAILED 110_Balanced_Binary_Tree/test_110_Balanced_Binary_Tree.py::test_solution_single_node_tree
FAILED 110_Balanced_Binary_Tree/test_110_Balanced_Binary_Tree.py::test_solution_two_node_tree
FAILED 110_Balanced_Binary_Tree/test_110_Balanced_Binary_Tree.py::test_solution_two_node_unbalanced_tree
========================= 6 failed, 3 passed in 0.02s ==========================

******* 111_Minimum_Depth_of_Binary_Tree ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/111_Minimum_Depth_of_Binary_Tree/test_111_Minimum_Depth_of_Binary_Tree.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, TreeNode
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/111_Minimum_Depth_of_Binary_Tree/solution.py)

******* 112_Path_Sum ***********

============================== 1 passed in 0.00s ===============================

******* 113_Path_Sum_II ***********

========================= 4 failed, 3 passed in 0.03s ==========================
FAILED 113_Path_Sum_II/test_113_Path_Sum_II.py::test_pathSum_positive_num - a...
FAILED 113_Path_Sum_II/test_113_Path_Sum_II.py::test_pathSum_negative_num - a...
FAILED 113_Path_Sum_II/test_113_Path_Sum_II.py::test_pathSum_mixed - assert [...
FAILED 113_Path_Sum_II/test_113_Path_Sum_II.py::test_pathSum_zero - assert [[...
========================= 4 failed, 3 passed in 0.03s ==========================

******* 114_Flatten_Binary_Tree_to_Linked_List ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/114_Flatten_Binary_Tree_to_Linked_List/test_114_Flatten_Binary_Tree_to_Linked_List.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import TreeNode, Solution
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/114_Flatten_Binary_Tree_to_Linked_List/solution.py)

******* 115_Distinct_Subsequences ***********

FAILED 115_Distinct_Subsequences/test_115_Distinct_Subsequences.py::test_numDistinct
============================== 1 failed in 0.01s ===============================

******* 116_Populating_Next_Right_Pointers_in_Each_Node ***********

============================== 3 passed in 0.01s ===============================

******* 117_Populating_Next_Right_Pointers_in_Each_Node_II ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/117_Populating_Next_Right_Pointers_in_Each_Node_II/test_117_Populating_Next_Right_Pointers_in_Each_Node_II.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, TreeLinkNode
E   ImportError: cannot import name 'TreeLinkNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/117_Populating_Next_Right_Pointers_in_Each_Node_II/solution.py)

******* 118_Pascal's_Triangle ***********

========================= 1 failed, 5 passed in 0.02s ==========================
E       Failed: DID NOT RAISE <class 'ValueError'>
118_Pascal's_Triangle/test_118_Pascal's_Triangle.py:43: Failed
FAILED 118_Pascal's_Triangle/test_118_Pascal's_Triangle.py::test_generate_negative_rows_raises_value_error
========================= 1 failed, 5 passed in 0.02s ==========================

******* 119_Pascal's_Triangle_II ***********

============================== 1 passed in 0.00s ===============================

******* 120_Triangle ***********

========================= 1 failed, 1 passed in 0.02s ==========================
FAILED 120_Triangle/test_120_Triangle.py::Tests::test_minimumTotal_returns_minimum_sum_of_triangle
========================= 1 failed, 1 passed in 0.02s ==========================

******* 121_Best_Time_to_Buy_and_Sell_Stock ***********

============================== 1 passed in 0.00s ===============================

******* 122_Best_Time_to_Buy_and_Sell_Stock_II ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked

******* 123_Best_Time_to_Buy_and_Sell_Stock_III ***********

FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_empty_prices
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_single_price
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_increasing_prices
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_decreasing_prices
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_alternating_prices_1
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_alternating_prices_2
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_random_prices_1
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_random_prices_2
FAILED 123_Best_Time_to_Buy_and_Sell_Stock_III/test_123_Best_Time_to_Buy_and_Sell_Stock_III.py::TestSolution::test_random_prices_3
============================== 9 failed in 0.03s ===============================

******* 124_Binary_Tree_Maximum_Path_Sum ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/124_Binary_Tree_Maximum_Path_Sum/test_124_Binary_Tree_Maximum_Path_Sum.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, TreeNode
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/124_Binary_Tree_Maximum_Path_Sum/solution.py)

******* 125_Valid_Palindrome ***********


******* 1260_Shift_2D_Grid ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/1260_Shift_2D_Grid/test_1260_Shift_2D_Grid.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution_file import Solution
E   ImportError: attempted relative import with no known parent package

******* 126_Word_Ladder_II ***********

FAILED 126_Word_Ladder_II/test_126_Word_Ladder_II.py::test_findLadders - Asse...
============================== 1 failed in 0.01s ===============================

******* 127_Word_Ladder ***********


******* 128_Longest_Consecutive_Sequence ***********

FAILED 128_Longest_Consecutive_Sequence/test_128_Longest_Consecutive_Sequence.py::test_longest_consecutive
============================== 1 failed in 0.01s ===============================

******* 1290_Convert_Binary_Number_in_a_Linked_List_to_Integer ***********

FAILED 1290_Convert_Binary_Number_in_a_Linked_List_to_Integer/test_1290_Convert_Binary_Number_in_a_Linked_List_to_Integer.py::test_getDecimalValue
============================== 1 failed in 0.01s ===============================

******* 129_Sum_Root_to_Leaf_Numbers ***********

FAILED 129_Sum_Root_to_Leaf_Numbers/test_129_Sum_Root_to_Leaf_Numbers.py::test_sumNumbers
============================== 1 failed in 0.01s ===============================

******* 1304_Find_N_Unique_Integers_Sum_up_to_Zero ***********

    from solution import Solution

******* 130_Surrounded_Regions ***********

============================== 1 passed in 0.00s ===============================

******* 1310_XOR_Queries_of_a_Subarray ***********

    from solution import Solution

******* 131_Palindrome_Partitioning ***********

============================== 2 passed in 0.00s ===============================

******* 1323_Maximum_69_Number ***********

========================= 3 failed, 5 passed in 0.02s ==========================
E       Failed: DID NOT RAISE <class 'ValueError'>
1323_Maximum_69_Number/test_1323_Maximum_69_Number.py:38: Failed
E       Failed: DID NOT RAISE <class 'TypeError'>
1323_Maximum_69_Number/test_1323_Maximum_69_Number.py:44: Failed
FAILED 1323_Maximum_69_Number/test_1323_Maximum_69_Number.py::test_no_6 - Ass...
FAILED 1323_Maximum_69_Number/test_1323_Maximum_69_Number.py::test_negative_input
FAILED 1323_Maximum_69_Number/test_1323_Maximum_69_Number.py::test_non_integer_input
========================= 3 failed, 5 passed in 0.02s ==========================

******* 132_Palindrome_Partitioning_II ***********

FAILED 132_Palindrome_Partitioning_II/test_132_Palindrome_Partitioning_II.py::test_solution
============================== 1 failed in 0.01s ===============================

******* 1337_The_K_Weakest_Rows_in_a_Matrix ***********


******* 133_Clone_Graph ***********

============================== 3 passed in 0.01s ===============================

******* 1342_Number_of_Steps_to_Reduce_a_Number_to_Zero ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked
    from solution import Solution

******* 134_Gas_Station ***********

========================= 4 failed, 1 passed in 0.02s ==========================
FAILED 134_Gas_Station/test_134_Gas_Station.py::test_canCompleteCircuit_empty
FAILED 134_Gas_Station/test_134_Gas_Station.py::test_canCompleteCircuit_no_solution
FAILED 134_Gas_Station/test_134_Gas_Station.py::test_canCompleteCircuit_solution
FAILED 134_Gas_Station/test_134_Gas_Station.py::test_canCompleteCircuit_large_input
========================= 4 failed, 1 passed in 0.02s ==========================

******* 135_Candy ***********

========================= 2 failed, 3 passed in 0.02s ==========================
FAILED 135_Candy/test_135_Candy.py::test_candy_with_two_elements - assert 3 == 2
FAILED 135_Candy/test_135_Candy.py::test_candy_with_multiple_elements - asser...
========================= 2 failed, 3 passed in 0.02s ==========================

******* 1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number/test_1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution
E   ImportError: attempted relative import with no known parent package

******* 136_Single_Number ***********

FAILED 136_Single_Number/test_136_Single_Number.py::TestSolution::test_singleNumber_with_hash
FAILED 136_Single_Number/test_136_Single_Number.py::TestSolution::test_singleNumber_with_set
FAILED 136_Single_Number/test_136_Single_Number.py::TestSolution::test_singleNumber_with_xor
FAILED 136_Single_Number/test_136_Single_Number.py::TestSolution::test_singleNumber_with_empty
FAILED 136_Single_Number/test_136_Single_Number.py::TestSolution::test_singleNumber_with_one_element
============================== 5 failed in 0.02s ===============================

******* 1374_Generate_a_String_With_Characters_That_Have_Odd_Counts_Solution ***********

========================= 1 failed, 5 passed in 0.02s ==========================
FAILED 1374_Generate_a_String_With_Characters_That_Have_Odd_Counts_Solution/test_1374_Generate_a_String_With_Characters_That_Have_Odd_Counts_Solution.py::test_generateTheString_6
========================= 1 failed, 5 passed in 0.02s ==========================

******* 137_Single_Number_II ***********

============================== 2 passed in 0.00s ===============================

******* 138_Copy_List_with_Random_Pointer ***********


******* 139_Word_Break ***********

============================== 1 passed in 0.00s ===============================

******* 140_Word_Break_II ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/140_Word_Break_II/test_140_Word_Break_II.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .filename import Solution
E   ImportError: attempted relative import with no known parent package

******* 141_Linked_List_Cycle ***********

============================== 11 passed in 0.01s ==============================

******* 142_Linked_List_Cycle_II ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked

******* 143_Reorder_List ***********

FAILED 143_Reorder_List/test_143_Reorder_List.py::test_reorderList_with_empty_list
FAILED 143_Reorder_List/test_143_Reorder_List.py::test_reorderList_with_single_node_list
FAILED 143_Reorder_List/test_143_Reorder_List.py::test_reorderList_with_two_node_list
FAILED 143_Reorder_List/test_143_Reorder_List.py::test_reorderList_with_odd_number_node_list
FAILED 143_Reorder_List/test_143_Reorder_List.py::test_reorderList_with_even_number_node_list
============================== 5 failed in 0.03s ===============================

******* 144_Binary_Tree_Preorder_Traversal ***********

============================== 1 passed in 0.00s ===============================

******* 145_Binary_Tree_Postorder_Traversal ***********

FAILED 145_Binary_Tree_Postorder_Traversal/test_145_Binary_Tree_Postorder_Traversal.py::TestSolution::test_empty_tree
FAILED 145_Binary_Tree_Postorder_Traversal/test_145_Binary_Tree_Postorder_Traversal.py::TestSolution::test_single_node_tree
FAILED 145_Binary_Tree_Postorder_Traversal/test_145_Binary_Tree_Postorder_Traversal.py::TestSolution::test_full_binary_tree
FAILED 145_Binary_Tree_Postorder_Traversal/test_145_Binary_Tree_Postorder_Traversal.py::TestSolution::test_left_skewed_tree
FAILED 145_Binary_Tree_Postorder_Traversal/test_145_Binary_Tree_Postorder_Traversal.py::TestSolution::test_right_skewed_tree
============================== 5 failed in 0.02s ===============================

******* 146_LRU_Cache ***********


******* 147_Insertion_Sort_List ***********

FAILED 147_Insertion_Sort_List/test_147_Insertion_Sort_List.py::test_insertionSortList
============================== 1 failed in 0.01s ===============================

******* 1480_Running_Sum_of_1d_Array ***********

FAILED 1480_Running_Sum_of_1d_Array/test_1480_Running_Sum_of_1d_Array.py::test_runningSum_empty
FAILED 1480_Running_Sum_of_1d_Array/test_1480_Running_Sum_of_1d_Array.py::test_runningSum_single_value
FAILED 1480_Running_Sum_of_1d_Array/test_1480_Running_Sum_of_1d_Array.py::test_runningSum_multiple_values
============================== 3 failed in 0.01s ===============================

******* 149_Max_Points_on_a_Line ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/149_Max_Points_on_a_Line/test_149_Max_Points_on_a_Line.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, Point
E   ImportError: cannot import name 'Point' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/149_Max_Points_on_a_Line/solution.py)

******* 150_Evaluate_Reverse_Polish_Notation ***********

FAILED 150_Evaluate_Reverse_Polish_Notation/test_150_Evaluate_Reverse_Polish_Notation.py::test_evalRPN
============================== 1 failed in 0.01s ===============================

******* 151_Reverse_Words_in_a_String ***********

FAILED 151_Reverse_Words_in_a_String/test_151_Reverse_Words_in_a_String.py::test_reverseWords_empty_string
FAILED 151_Reverse_Words_in_a_String/test_151_Reverse_Words_in_a_String.py::test_reverseWords_only_spaces
FAILED 151_Reverse_Words_in_a_String/test_151_Reverse_Words_in_a_String.py::test_reverseWords_single_word
FAILED 151_Reverse_Words_in_a_String/test_151_Reverse_Words_in_a_String.py::test_reverseWords_multiple_words
FAILED 151_Reverse_Words_in_a_String/test_151_Reverse_Words_in_a_String.py::test_reverseWords_spaces_between_words
FAILED 151_Reverse_Words_in_a_String/test_151_Reverse_Words_in_a_String.py::test_reverseWords_complex_string
============================== 6 failed in 0.03s ===============================

******* 152_Maximum_Product_Subarray ***********

========================= 1 failed, 6 passed in 0.02s ==========================
FAILED 152_Maximum_Product_Subarray/test_152_Maximum_Product_Subarray.py::test_maxProduct_with_mixed_num
========================= 1 failed, 6 passed in 0.02s ==========================

******* 153_Find_Minimum_in_Rotated_Sorted_Array ***********

FAILED 153_Find_Minimum_in_Rotated_Sorted_Array/test_153_Find_Minimum_in_Rotated_Sorted_Array.py::test_findMin
============================== 1 failed in 0.01s ===============================

******* 154_Find_Minimum_in_Rotated_Sorted_Array_II ***********

FAILED 154_Find_Minimum_in_Rotated_Sorted_Array_II/test_154_Find_Minimum_in_Rotated_Sorted_Array_II.py::test_findMin
============================== 1 failed in 0.02s ===============================

******* 155_Min_Stack ***********

FAILED 155_Min_Stack/test_155_Min_Stack.py::test_push - NameError: name 'MinS...
FAILED 155_Min_Stack/test_155_Min_Stack.py::test_pop - NameError: name 'MinSt...
FAILED 155_Min_Stack/test_155_Min_Stack.py::test_top - NameError: name 'MinSt...
FAILED 155_Min_Stack/test_155_Min_Stack.py::test_getMin - NameError: name 'Mi...
============================== 4 failed in 0.02s ===============================

******* 156_Binary_Tree_Upside_Down ***********

========================= 5 failed, 1 passed in 0.02s ==========================
FAILED 156_Binary_Tree_Upside_Down/test_156_Binary_Tree_Upside_Down.py::TestSolution::test_upsideDownBinaryTree_singleNode
FAILED 156_Binary_Tree_Upside_Down/test_156_Binary_Tree_Upside_Down.py::TestSolution::test_upsideDownBinaryTree_twoNodes
FAILED 156_Binary_Tree_Upside_Down/test_156_Binary_Tree_Upside_Down.py::TestSolution::test_upsideDownBinaryTree_threeNodes
FAILED 156_Binary_Tree_Upside_Down/test_156_Binary_Tree_Upside_Down.py::TestSolution::test_upsideDownBinaryTree_balanced
FAILED 156_Binary_Tree_Upside_Down/test_156_Binary_Tree_Upside_Down.py::TestSolution::test_upsideDownBinaryTree_unbalanced
========================= 5 failed, 1 passed in 0.02s ==========================

******* 157_Read_N_Characters_Given_Read4 ***********

FAILED 157_Read_N_Characters_Given_Read4/test_157_Read_N_Characters_Given_Read4.py::TestSolution::test_read1
FAILED 157_Read_N_Characters_Given_Read4/test_157_Read_N_Characters_Given_Read4.py::TestSolution::test_read2
FAILED 157_Read_N_Characters_Given_Read4/test_157_Read_N_Characters_Given_Read4.py::TestSolution::test_read3
FAILED 157_Read_N_Characters_Given_Read4/test_157_Read_N_Characters_Given_Read4.py::TestSolution::test_read4
FAILED 157_Read_N_Characters_Given_Read4/test_157_Read_N_Characters_Given_Read4.py::TestSolution::test_read5
============================== 5 failed in 0.02s ===============================

******* 158_Read_N_Characters_Given_Read4_II_Call_multiple_times ***********

FAILED 158_Read_N_Characters_Given_Read4_II_Call_multiple_times/test_158_Read_N_Characters_Given_Read4_II_Call_multiple_times.py::test_read_empty_buf
FAILED 158_Read_N_Characters_Given_Read4_II_Call_multiple_times/test_158_Read_N_Characters_Given_Read4_II_Call_multiple_times.py::test_read_zero_characters
FAILED 158_Read_N_Characters_Given_Read4_II_Call_multiple_times/test_158_Read_N_Characters_Given_Read4_II_Call_multiple_times.py::test_read_less_than_4_characters
FAILED 158_Read_N_Characters_Given_Read4_II_Call_multiple_times/test_158_Read_N_Characters_Given_Read4_II_Call_multiple_times.py::test_read_multiple_times
FAILED 158_Read_N_Characters_Given_Read4_II_Call_multiple_times/test_158_Read_N_Characters_Given_Read4_II_Call_multiple_times.py::test_read_more_than_available
============================== 5 failed in 0.02s ===============================

******* 1599_Maximum_Profit_of_Operating_a_Centennial_Wheel ***********

========================= 1 failed, 4 passed in 0.02s ==========================
FAILED 1599_Maximum_Profit_of_Operating_a_Centennial_Wheel/test_1599_Maximum_Profit_of_Operating_a_Centennial_Wheel.py::test_minOperationsMaxProfit5
========================= 1 failed, 4 passed in 0.02s ==========================

******* 159_Longest_Substring_with_At_Most_Two_Distinct_Characters ***********

FAILED 159_Longest_Substring_with_At_Most_Two_Distinct_Characters/test_159_Longest_Substring_with_At_Most_Two_Distinct_Characters.py::test_lengthOfLongestSubstringTwoDistinct
============================== 1 failed in 0.01s ===============================

******* 160_Intersection_of_Two_Linked_Lists ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/160_Intersection_of_Two_Linked_Lists/test_160_Intersection_of_Two_Linked_Lists.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import ListNode, Solution
E   ImportError: cannot import name 'ListNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/160_Intersection_of_Two_Linked_Lists/solution.py)

******* 161_One_Edit_Distance ***********

============================== 10 passed in 0.01s ==============================

******* 162_Find_Peak_Element ***********

FAILED 162_Find_Peak_Element/test_162_Find_Peak_Element.py::test_findPeakElement
============================== 1 failed in 0.01s ===============================

******* 163_Missing_Ranges ***********

========================= 1 failed, 4 passed in 0.02s ==========================
FAILED 163_Missing_Ranges/test_163_Missing_Ranges.py::test_findMissingRanges_full
========================= 1 failed, 4 passed in 0.02s ==========================

******* 165_Compare_Version_Numbers ***********

FAILED 165_Compare_Version_Numbers/test_165_Compare_Version_Numbers.py::TestSolution::test_compareVersion
============================== 1 failed in 0.01s ===============================

******* 166_Fraction_to_Recurring_Decimal ***********

========================= 3 failed, 2 passed in 0.02s ==========================
FAILED 166_Fraction_to_Recurring_Decimal/test_166_Fraction_to_Recurring_Decimal.py::test_fraction_to_decimal_positive
FAILED 166_Fraction_to_Recurring_Decimal/test_166_Fraction_to_Recurring_Decimal.py::test_fraction_to_decimal_negative
FAILED 166_Fraction_to_Recurring_Decimal/test_166_Fraction_to_Recurring_Decimal.py::test_fraction_to_decimal_recurring
========================= 3 failed, 2 passed in 0.02s ==========================

******* 167_Two_Sum_II_Input_array_is_sorted ***********

FAILED 167_Two_Sum_II_Input_array_is_sorted/test_167_Two_Sum_II_Input_array_is_sorted.py::test_twoSum_positive_numbers
FAILED 167_Two_Sum_II_Input_array_is_sorted/test_167_Two_Sum_II_Input_array_is_sorted.py::test_twoSum_zero_target
FAILED 167_Two_Sum_II_Input_array_is_sorted/test_167_Two_Sum_II_Input_array_is_sorted.py::test_twoSum_negative_numbers
FAILED 167_Two_Sum_II_Input_array_is_sorted/test_167_Two_Sum_II_Input_array_is_sorted.py::test_twoSum_duplicate_numbers
============================== 4 failed in 0.02s ===============================

******* 168_Excel_Sheet_Column_Title ***********

============================== 5 passed in 0.01s ===============================

******* 170_Two_Sum_III-Data_structure_design ***********

FAILED 170_Two_Sum_III-Data_structure_design/test_170_Two_Sum_III-Data_structure_design.py::test_add
FAILED 170_Two_Sum_III-Data_structure_design/test_170_Two_Sum_III-Data_structure_design.py::test_find_true
FAILED 170_Two_Sum_III-Data_structure_design/test_170_Two_Sum_III-Data_structure_design.py::test_find_false
============================== 3 failed in 0.01s ===============================

******* 179_Largest_Number ***********

============================== 1 passed in 0.00s ===============================

******* 186_Reverse_Words_in_a_String_II ***********

FAILED 186_Reverse_Words_in_a_String_II/test_186_Reverse_Words_in_a_String_II.py::test_reverseWords
============================== 1 failed in 0.01s ===============================

******* 1909_Remove_One_Element_to_Make_the_Array_Strictly_Increasing ***********

    from solution import Solution

******* 191_Number_of_1_Bits ***********

============================== 1 passed in 0.00s ===============================

******* 198_House_Robber ***********

========================= 1 failed, 4 passed in 0.01s ==========================
FAILED 198_House_Robber/test_198_House_Robber.py::test_rob_with_negative_numbers
========================= 1 failed, 4 passed in 0.01s ==========================

******* 200_Number_of_Islands ***********

============================== 1 passed in 0.00s ===============================

******* 202_Happy_Number ***********

========================= 1 failed, 1 passed in 0.02s ==========================
FAILED 202_Happy_Number/test_202_Happy_Number.py::test_happy_number - ValueEr...
========================= 1 failed, 1 passed in 0.02s ==========================

******* 203_Remove_Linked_List_Elements ***********

FAILED 203_Remove_Linked_List_Elements/test_203_Remove_Linked_List_Elements.py::test_removeElements_empty_list
FAILED 203_Remove_Linked_List_Elements/test_203_Remove_Linked_List_Elements.py::test_removeElements_remove_all_elements
FAILED 203_Remove_Linked_List_Elements/test_203_Remove_Linked_List_Elements.py::test_removeElements_remove_head
FAILED 203_Remove_Linked_List_Elements/test_203_Remove_Linked_List_Elements.py::test_removeElements_remove_middle_element
FAILED 203_Remove_Linked_List_Elements/test_203_Remove_Linked_List_Elements.py::test_removeElements_remove_last_element
FAILED 203_Remove_Linked_List_Elements/test_203_Remove_Linked_List_Elements.py::test_removeElements_no_element_to_remove
============================== 6 failed in 0.02s ===============================

******* 204_Count_Primes ***********

E       Failed: DID NOT RAISE <class 'ValueError'>
204_Count_Primes/test_204_Count_Primes.py:16: Failed
FAILED 204_Count_Primes/test_204_Count_Primes.py::test_countPrimes - Failed: ...
============================== 1 failed in 0.01s ===============================

******* 205_Isomorphic_Strings ***********

============================== 2 passed in 0.00s ===============================

******* 206_Reverse_Linked_List ***********

FAILED 206_Reverse_Linked_List/test_206_Reverse_Linked_List.py::test_reverseList_iteratively
FAILED 206_Reverse_Linked_List/test_206_Reverse_Linked_List.py::test_reverseList_recursively
============================== 2 failed in 0.01s ===============================

******* 207_Course_Schedule ***********

============================== 1 passed in 0.00s ===============================

******* 208_Implement_Trie_(Prefix_Tree) ***********

FAILED 208_Implement_Trie_(Prefix_Tree)/test_208_Implement_Trie_(Prefix_Tree).py::test_trie_node_creation
========================= 1 failed, 4 errors in 0.02s ==========================

******* 213_House_Robber_II ***********

========================= 2 failed, 10 passed in 0.02s =========================
FAILED 213_House_Robber_II/test_213_House_Robber_II.py::test_rob[nums1-12] - ...
FAILED 213_House_Robber_II/test_213_House_Robber_II.py::test_rob[nums2-4] - a...
========================= 2 failed, 10 passed in 0.02s =========================

******* 215_Kth_Largest_Element_in_an_Array ***********

FAILED 215_Kth_Largest_Element_in_an_Array/test_215_Kth_Largest_Element_in_an_Array.py::test_findKthLargest
============================== 1 failed in 0.01s ===============================

******* 216_Combination_Sum_III ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked
    from solution import Solution

******* 217_Contains_Duplicate ***********

============================== 1 passed in 0.00s ===============================

******* 219_Contains_Duplicate_II ***********

============================== 7 passed in 0.01s ===============================

******* 220_Contains_Duplicate_III ***********

FAILED 220_Contains_Duplicate_III/test_220_Contains_Duplicate_III.py::test_containsNearbyAlmostDuplicate
============================== 1 failed in 0.02s ===============================

******* 221_Maximal_Square ***********

FAILED 221_Maximal_Square/test_221_Maximal_Square.py::test_maximalSquare_bruteForce
FAILED 221_Maximal_Square/test_221_Maximal_Square.py::test_maximalSquare_dp
FAILED 221_Maximal_Square/test_221_Maximal_Square.py::test_maximalSquare_dp_nSpace
============================== 3 failed in 0.02s ===============================

******* 223_Rectangle ***********


******* 223_Rectangle Area ***********

FAILED 223_Rectangle Area/test_223_Rectangle Area.py::test_computeArea - asse...
============================== 1 failed in 0.01s ===============================

******* 225_Implement_Stack_using_Queues ***********

FAILED 225_Implement_Stack_using_Queues/test_225_Implement_Stack_using_Queues.py::test_push
FAILED 225_Implement_Stack_using_Queues/test_225_Implement_Stack_using_Queues.py::test_pop
FAILED 225_Implement_Stack_using_Queues/test_225_Implement_Stack_using_Queues.py::test_top
FAILED 225_Implement_Stack_using_Queues/test_225_Implement_Stack_using_Queues.py::test_empty
============================== 4 failed in 0.02s ===============================

******* 226_Invert_Binary_Tree ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/226_Invert_Binary_Tree/test_226_Invert_Binary_Tree.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution_file import Solution, TreeNode
E   ImportError: attempted relative import with no known parent package

******* 228_Summary_Ranges ***********

========================= 2 failed, 6 passed in 0.02s ==========================
FAILED 228_Summary_Ranges/test_228_Summary_Ranges.py::test_summaryRanges_duplicate
FAILED 228_Summary_Ranges/test_228_Summary_Ranges.py::test_summaryRanges_all_same
========================= 2 failed, 6 passed in 0.02s ==========================

******* 229_Majority_Element_II ***********

FAILED 229_Majority_Element_II/test_229_Majority_Element_II.py::test_majorityElement
============================== 1 failed in 0.01s ===============================

******* 231_Power_of_Two ***********

============================== 3 passed in 0.01s ===============================

******* 232_Implement_Queue_using_Stacks ***********

FAILED 232_Implement_Queue_using_Stacks/test_232_Implement_Queue_using_Stacks.py::test_push
FAILED 232_Implement_Queue_using_Stacks/test_232_Implement_Queue_using_Stacks.py::test_pop
FAILED 232_Implement_Queue_using_Stacks/test_232_Implement_Queue_using_Stacks.py::test_peek
FAILED 232_Implement_Queue_using_Stacks/test_232_Implement_Queue_using_Stacks.py::test_empty
============================== 4 failed in 0.02s ===============================

******* 234_Palindrome_Linked_List ***********

========================= 9 failed, 1 passed in 0.04s ==========================
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_single_valued_list
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_two_valued_list_different
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_two_valued_list_same
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_three_valued_list_palindrome
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_three_valued_list_not_palindrome
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_four_valued_list_palindrome
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_four_valued_list_not_palindrome
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_five_valued_list_palindrome
FAILED 234_Palindrome_Linked_List/test_234_Palindrome_Linked_List.py::test_five_valued_list_not_palindrome
========================= 9 failed, 1 passed in 0.04s ==========================

******* 235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree ***********

FAILED 235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree/test_235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree.py::test_lowestCommonAncestor1
FAILED 235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree/test_235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree.py::test_lowestCommonAncestor2
FAILED 235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree/test_235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree.py::test_lowestCommonAncestor3
FAILED 235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree/test_235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree.py::test_lowestCommonAncestor4
FAILED 235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree/test_235_Lowest_Common_Ancestor_of_a_Binary_Search_Tree.py::test_lowestCommonAncestor5
============================== 5 failed in 0.02s ===============================

******* 236_Lowest_Common_Ancestor_of_a_Binary_Tree ***********

============================== 1 passed in 0.00s ===============================

******* 237_Delete_Node_in_a_Linked_List ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/237_Delete_Node_in_a_Linked_List/test_237_Delete_Node_in_a_Linked_List.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import ListNode, Solution
E   ImportError: cannot import name 'ListNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/237_Delete_Node_in_a_Linked_List/solution.py)

******* 238_Product_of_Array_Except_Self ***********

============================== 1 passed in 0.00s ===============================

******* 2409_Count_Days_Spent_Together ***********

FAILED 2409_Count_Days_Spent_Together/test_2409_Count_Days_Spent_Together.py::test_countDaysTogether
============================== 1 failed in 0.01s ===============================

******* 2413_Smallest_Even_Multiple ***********

FAILED 2413_Smallest_Even_Multiple/test_2413_Smallest_Even_Multiple.py::test_smallestEvenMultiple
============================== 1 failed in 0.01s ===============================

******* 2420_Find_All_Good_Indices ***********


******* 2429_Minimize_XOR ***********


******* 242_Valid_Anagram ***********

============================== 7 passed in 0.01s ===============================

******* 243_Shortest_Word_Distance ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/243_Shortest_Word_Distance/test_243_Shortest_Word_Distance.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution
E   ImportError: attempted relative import with no known parent package

******* 246_Strobogrammatic_Number ***********

============================== 1 passed in 0.00s ===============================

******* 249_Group_Shifted_Strings ***********


******* 252_Meeting_Rooms ***********


******* 253_Meeting_Rooms_II ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/253_Meeting_Rooms_II/test_253_Meeting_Rooms_II.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Interval, Solution
E   ImportError: cannot import name 'Interval' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/253_Meeting_Rooms_II/solution.py)

******* 257_Binary_Tree_Paths ***********

FAILED 257_Binary_Tree_Paths/test_257_Binary_Tree_Paths.py::TestBinaryTreePaths::test_empty_tree
FAILED 257_Binary_Tree_Paths/test_257_Binary_Tree_Paths.py::TestBinaryTreePaths::test_single_node_tree
FAILED 257_Binary_Tree_Paths/test_257_Binary_Tree_Paths.py::TestBinaryTreePaths::test_multiple_nodes_tree
============================== 3 failed in 0.01s ===============================

******* 258_Add_Digits ***********

FAILED 258_Add_Digits/test_258_Add_Digits.py::test_addDigits - assert 1.0 == 2
============================== 1 failed in 0.01s ===============================

******* 259_3Sum_Smaller ***********

FAILED 259_3Sum_Smaller/test_259_3Sum_Smaller.py::TestSolution::test_threeSumSmaller
FAILED 259_3Sum_Smaller/test_259_3Sum_Smaller.py::TestSolution::test_twoSumSmaller
FAILED 259_3Sum_Smaller/test_259_3Sum_Smaller.py::TestSolution::test_binarySearch
============================== 3 failed in 0.02s ===============================

******* 263_Ugly_Number ***********

========================= 2 failed, 2 passed in 0.02s ==========================
FAILED 263_Ugly_Number/test_263_Ugly_Number.py::test_is_ugly_returns_true_for_ugly_numbers
FAILED 263_Ugly_Number/test_263_Ugly_Number.py::test_is_ugly_returns_false_for_non_ugly_numbers
========================= 2 failed, 2 passed in 0.02s ==========================

******* 264_Ugly_Number_II ***********

FAILED 264_Ugly_Number_II/test_264_Ugly_Number_II.py::TestSolution::test_n_th_ugly_number
============================== 1 failed in 0.01s ===============================

******* 266_Palindrome_Permutation ***********

============================== 6 passed in 0.01s ===============================

******* 267_Palindrome_Permutation_II ***********

========================= 2 failed, 3 passed in 0.02s ==========================
FAILED 267_Palindrome_Permutation_II/test_267_Palindrome_Permutation_II.py::test_generatePalindromes_returns_correct_output
FAILED 267_Palindrome_Permutation_II/test_267_Palindrome_Permutation_II.py::test_generatePalindromes_returns_empty_list_if_odd_count_of_chars_greater_than_1
========================= 2 failed, 3 passed in 0.02s ==========================

******* 268_Missing_Number ***********

FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_1 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_2 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_3 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_4 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_5 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_6 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_7 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_8 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_9 - ...
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_10
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_11
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_12
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_13
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_14
FAILED 268_Missing_Number/test_268_Missing_Number.py::test_missingNumber_15
============================== 15 failed in 0.04s ==============================

******* 270_Closest_Binary_Search_Tree_Value ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/270_Closest_Binary_Search_Tree_Value/test_270_Closest_Binary_Search_Tree_Value.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, TreeNode
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/270_Closest_Binary_Search_Tree_Value/solution.py)

******* 273_Integer_to_English_Words ***********

FAILED 273_Integer_to_English_Words/test_273_Integer_to_English_Words.py::test_numberToWords
============================== 1 failed in 0.01s ===============================

******* 274_H-Index ***********

============================== 2 passed in 0.00s ===============================

******* 276_Paint_Fence ***********

============================== 5 passed in 0.01s ===============================

******* 278_First_Bad_Version ***********

========================= 1 failed, 1 passed in 0.01s ==========================
FAILED 278_First_Bad_Version/test_278_First_Bad_Version.py::TestSolution::test_firstBadVersion
========================= 1 failed, 1 passed in 0.01s ==========================

******* 280_Wiggle_Sort ***********

FAILED 280_Wiggle_Sort/test_280_Wiggle_Sort.py::TestWiggleSort::test_wiggle_sort_empty_list
FAILED 280_Wiggle_Sort/test_280_Wiggle_Sort.py::TestWiggleSort::test_wiggle_sort_one_element_list
FAILED 280_Wiggle_Sort/test_280_Wiggle_Sort.py::TestWiggleSort::test_wiggle_sort_sorted_list
FAILED 280_Wiggle_Sort/test_280_Wiggle_Sort.py::TestWiggleSort::test_wiggle_sort_reversed_list
FAILED 280_Wiggle_Sort/test_280_Wiggle_Sort.py::TestWiggleSort::test_wiggle_sort_random_list
============================== 5 failed in 0.02s ===============================

******* 283_Move ***********


******* 283_Move Zeroes ***********

FAILED 283_Move Zeroes/test_283_Move Zeroes.py::TestSolution::test_moveZeroes1
FAILED 283_Move Zeroes/test_283_Move Zeroes.py::TestSolution::test_moveZeroes2
FAILED 283_Move Zeroes/test_283_Move Zeroes.py::TestSolution::test_moveZeroes3
FAILED 283_Move Zeroes/test_283_Move Zeroes.py::TestSolution::test_moveZeroes4
FAILED 283_Move Zeroes/test_283_Move Zeroes.py::TestSolution::test_moveZeroes5
============================== 5 failed in 0.02s ===============================

******* 286_Walls_and_Gates ***********

============================== 6 passed in 0.01s ===============================

******* 288_Unique_Word_Abbreviation ***********


******* 290_Word_Pattern ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/290_Word_Pattern/test_290_Word_Pattern.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution
E   ImportError: attempted relative import with no known parent package

******* 292_Nim_Game ***********

========================= 1 failed, 3 passed in 0.01s ==========================
E       Failed: DID NOT RAISE <class 'ValueError'>
292_Nim_Game/test_292_Nim_Game.py:24: Failed
FAILED 292_Nim_Game/test_292_Nim_Game.py::test_canWinNim_negative - Failed: D...
========================= 1 failed, 3 passed in 0.01s ==========================

******* 293_Flip_Game ***********

FAILED 293_Flip_Game/test_293_Flip_Game.py::test_generate_possible_next_moves
============================== 1 failed in 0.01s ===============================

******* 294_Flip_Game_II ***********

FAILED 294_Flip_Game_II/test_294_Flip_Game_II.py::test_Solution - AssertionEr...
============================== 1 failed in 0.01s ===============================

******* 296_Best_Meeting_Point ***********

========================= 2 failed, 2 passed in 0.02s ==========================
FAILED 296_Best_Meeting_Point/test_296_Best_Meeting_Point.py::test_minDistance1D
FAILED 296_Best_Meeting_Point/test_296_Best_Meeting_Point.py::test_minTotalDistance
========================= 2 failed, 2 passed in 0.02s ==========================

******* 298_Binary_Tree_Longest_Consecutive_Sequence ***********


******* 299_Bulls_and_Cows ***********


******* 303_Range_Sum_Query_Immutable ***********

FAILED 303_Range_Sum_Query_Immutable/test_303_Range_Sum_Query_Immutable.py::test_empty_array
========================= 1 failed, 2 errors in 0.02s ==========================

******* 304_Range_Sum_Query_2D_Immutable ***********

FAILED 304_Range_Sum_Query_2D_Immutable/test_304_Range_Sum_Query_2D_Immutable.py::test_empty_matrix
FAILED 304_Range_Sum_Query_2D_Immutable/test_304_Range_Sum_Query_2D_Immutable.py::test_single_element_matrix
FAILED 304_Range_Sum_Query_2D_Immutable/test_304_Range_Sum_Query_2D_Immutable.py::test_square_matrix
FAILED 304_Range_Sum_Query_2D_Immutable/test_304_Range_Sum_Query_2D_Immutable.py::test_rectangular_matrix
FAILED 304_Range_Sum_Query_2D_Immutable/test_304_Range_Sum_Query_2D_Immutable.py::test_negative_values
FAILED 304_Range_Sum_Query_2D_Immutable/test_304_Range_Sum_Query_2D_Immutable.py::test_matrix_with_zero_values
============================== 6 failed in 0.02s ===============================

******* 305_Number_of_Islands_II ***********

FAILED 305_Number_of_Islands_II/test_305_Number_of_Islands_II.py::test_num_islands
============================== 1 failed in 0.01s ===============================

******* 307_Range_Sum_Query_Mutable ***********

FAILED 307_Range_Sum_Query_Mutable/test_307_Range_Sum_Query_Mutable.py::test_num_array
FAILED 307_Range_Sum_Query_Mutable/test_307_Range_Sum_Query_Mutable.py::test_build_tree
FAILED 307_Range_Sum_Query_Mutable/test_307_Range_Sum_Query_Mutable.py::test_update
FAILED 307_Range_Sum_Query_Mutable/test_307_Range_Sum_Query_Mutable.py::test_sum_range
============================== 4 failed in 0.02s ===============================

******* 322_Coin_Change ***********

FAILED 322_Coin_Change/test_322_Coin_Change.py::test_coinChange - assert 54 =...
============================== 1 failed in 0.02s ===============================

******* 326_Power_of_Three ***********

FAILED 326_Power_of_Three/test_326_Power_of_Three.py::test_isPowerOfThree_true
FAILED 326_Power_of_Three/test_326_Power_of_Three.py::test_isPowerOfThree_false
============================== 2 failed in 0.01s ===============================

******* 328_Odd_Even_Linked_List ***********

FAILED 328_Odd_Even_Linked_List/test_328_Odd_Even_Linked_List.py::TestSolution::test_oddEvenList_empty
FAILED 328_Odd_Even_Linked_List/test_328_Odd_Even_Linked_List.py::TestSolution::test_oddEvenList_single
FAILED 328_Odd_Even_Linked_List/test_328_Odd_Even_Linked_List.py::TestSolution::test_oddEvenList_double
FAILED 328_Odd_Even_Linked_List/test_328_Odd_Even_Linked_List.py::TestSolution::test_oddEvenList_odd
FAILED 328_Odd_Even_Linked_List/test_328_Odd_Even_Linked_List.py::TestSolution::test_oddEvenList_even
============================== 5 failed in 0.02s ===============================

******* 336_Palindrome_Pairs ***********

FAILED 336_Palindrome_Pairs/test_336_Palindrome_Pairs.py::test_palindrome_pairs
============================== 1 failed in 0.01s ===============================

******* 337_House_Robber_III ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/337_House_Robber_III/test_337_House_Robber_III.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import TreeNode, Solution
E   ImportError: cannot import name 'TreeNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/337_House_Robber_III/solution.py)

******* 338_Counting_Bits ***********


******* 339_Nested_List_Weight_Sum ***********

FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_isInteger_returns_true_for_single_integer_ni
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_isInteger_returns_false_for_nested_list_ni
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_getInteger_returns_integer_for_single_integer_ni
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_getInteger_returns_none_for_nested_list_ni
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_getList_returns_list_for_nested_list_ni
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_getList_returns_none_for_single_integer_ni
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_depthSum_returns_correct_sum_for_single_integer_list
FAILED 339_Nested_List_Weight_Sum/test_339_Nested_List_Weight_Sum.py::test_depthSum_returns_correct_sum_for_nested_list
============================== 8 failed in 0.02s ===============================

******* 340_Longest_Substring_with_At_Most_K_Distinct_Characters ***********

FAILED 340_Longest_Substring_with_At_Most_K_Distinct_Characters/test_340_Longest_Substring_with_At_Most_K_Distinct_Characters.py::test_lengthOfLongestSubstringKDistinct
============================== 1 failed in 0.01s ===============================

******* 342_Power_of_Four ***********

FAILED 342_Power_of_Four/test_342_Power_of_Four.py::test_is_power_of_four_true
FAILED 342_Power_of_Four/test_342_Power_of_Four.py::test_is_power_of_four_false
============================== 2 failed in 0.01s ===============================

******* 344_Reverse_String ***********

FAILED 344_Reverse_String/test_344_Reverse_String.py::test_reverse_string - N...
============================== 1 failed in 0.01s ===============================

******* 345_Reverse_Vowels_of_a_String ***********

FAILED 345_Reverse_Vowels_of_a_String/test_345_Reverse_Vowels_of_a_String.py::test_reverseVowels
============================== 1 failed in 0.01s ===============================

******* 346_Moving_Average_from_Data_Stream ***********


******* 347_Top_K_Frequent_Elements ***********

FAILED 347_Top_K_Frequent_Elements/test_347_Top_K_Frequent_Elements.py::test_empty_list
FAILED 347_Top_K_Frequent_Elements/test_347_Top_K_Frequent_Elements.py::test_single_element_list
FAILED 347_Top_K_Frequent_Elements/test_347_Top_K_Frequent_Elements.py::test_duplicate_elements
FAILED 347_Top_K_Frequent_Elements/test_347_Top_K_Frequent_Elements.py::test_k_is_greater_than_number_of_unique_elements
FAILED 347_Top_K_Frequent_Elements/test_347_Top_K_Frequent_Elements.py::test_k_is_equal_to_number_of_unique_elements
FAILED 347_Top_K_Frequent_Elements/test_347_Top_K_Frequent_Elements.py::test_all_elements_are_the_same
============================== 6 failed in 0.02s ===============================

******* 349_Intersection_of_Two_Arrays ***********

============================== 5 passed in 0.01s ===============================

******* 350_Intersection_of_Two_Arrays_II ***********

========================= 1 failed, 9 passed in 0.02s ==========================
FAILED 350_Intersection_of_Two_Arrays_II/test_350_Intersection_of_Two_Arrays_II.py::test_case_4
========================= 1 failed, 9 passed in 0.02s ==========================

******* 351_Android_Unlock_Patterns ***********

========================= 2 failed, 1 passed in 0.02s ==========================
FAILED 351_Android_Unlock_Patterns/test_351_Android_Unlock_Patterns.py::test_numberOfPatterns
FAILED 351_Android_Unlock_Patterns/test_351_Android_Unlock_Patterns.py::test_calc_patterns
========================= 2 failed, 1 passed in 0.02s ==========================

******* 359_Logger_Rate_Limiter ***********

FAILED 359_Logger_Rate_Limiter/test_359_Logger_Rate_Limiter.py::test_shouldPrintMessage
============================== 1 failed in 0.01s ===============================

******* 366_Find_Leaves_of_Binary_Tree ***********

========================= 4 failed, 1 passed in 0.02s ==========================
FAILED 366_Find_Leaves_of_Binary_Tree/test_366_Find_Leaves_of_Binary_Tree.py::TestSolution::test_findLeaves_singleNode
FAILED 366_Find_Leaves_of_Binary_Tree/test_366_Find_Leaves_of_Binary_Tree.py::TestSolution::test_findLeaves_leftChildOnly
FAILED 366_Find_Leaves_of_Binary_Tree/test_366_Find_Leaves_of_Binary_Tree.py::TestSolution::test_findLeaves_rightChildOnly
FAILED 366_Find_Leaves_of_Binary_Tree/test_366_Find_Leaves_of_Binary_Tree.py::TestSolution::test_findLeaves_completeTree
========================= 4 failed, 1 passed in 0.02s ==========================

******* 367_Valid_Perfect_Square ***********

========================= 2 failed, 2 passed in 0.02s ==========================
FAILED 367_Valid_Perfect_Square/test_367_Valid_Perfect_Square.py::test_case1
FAILED 367_Valid_Perfect_Square/test_367_Valid_Perfect_Square.py::test_case3
========================= 2 failed, 2 passed in 0.02s ==========================

******* 368_Largest_Divisible_Subset ***********

FAILED 368_Largest_Divisible_Subset/test_368_Largest_Divisible_Subset.py::test_largestDivisibleSubset
============================== 1 failed in 0.01s ===============================

******* 369_Plus_One_Linked_List ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/369_Plus_One_Linked_List/test_369_Plus_One_Linked_List.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import Solution, ListNode
E   ImportError: cannot import name 'ListNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/369_Plus_One_Linked_List/solution.py)

******* 370_Range_Addition ***********

FAILED 370_Range_Addition/test_370_Range_Addition.py::test_getModifiedArray
============================== 1 failed in 0.01s ===============================

******* 371_Sum_of_Two_Integers ***********

============================== 1 passed in 0.01s ===============================

******* 372_Super_Pow ***********

FAILED 372_Super_Pow/test_372_Super_Pow.py::test_superPow - assert 711 == 2048
FAILED 372_Super_Pow/test_372_Super_Pow.py::test_powmod - assert 918 == 15625
============================== 2 failed in 0.01s ===============================

******* 373_Find_K_Pairs_with_Smallest_Sums ***********

FAILED 373_Find_K_Pairs_with_Smallest_Sums/test_373_Find_K_Pairs_with_Smallest_Sums.py::test_kSmallestPairs
============================== 1 failed in 0.01s ===============================

******* 374_Guess_Number_Higher_or_Lower ***********

FAILED 374_Guess_Number_Higher_or_Lower/test_374_Guess_Number_Higher_or_Lower.py::test_guessNumber
============================== 1 failed in 0.01s ===============================

******* 375_Guess_Number_Higher_or_Lower_II ***********

FAILED 375_Guess_Number_Higher_or_Lower_II/test_375_Guess_Number_Higher_or_Lower_II.py::test_top_down_dp
FAILED 375_Guess_Number_Higher_or_Lower_II/test_375_Guess_Number_Higher_or_Lower_II.py::test_bottom_up_dp
============================== 2 failed in 0.01s ===============================

******* 380_Insert_Delete_GetRandom ***********

FAILED 380_Insert_Delete_GetRandom/test_380_Insert_Delete_GetRandom.py::test_insert
FAILED 380_Insert_Delete_GetRandom/test_380_Insert_Delete_GetRandom.py::test_remove
FAILED 380_Insert_Delete_GetRandom/test_380_Insert_Delete_GetRandom.py::test_getRandom
FAILED 380_Insert_Delete_GetRandom/test_380_Insert_Delete_GetRandom.py::test_combined
============================== 4 failed in 0.02s ===============================

******* 383_Ransom_Note ***********

============================== 3 passed in 0.01s ===============================

******* 384_Shuffle_an_Array ***********

FAILED 384_Shuffle_an_Array/test_384_Shuffle_an_Array.py::test_reset - NameEr...
FAILED 384_Shuffle_an_Array/test_384_Shuffle_an_Array.py::test_shuffle - Name...
============================== 2 failed in 0.01s ===============================

******* 387_First_Unique_Character_in_a_String ***********

FAILED 387_First_Unique_Character_in_a_String/test_387_First_Unique_Character_in_a_String.py::test_firstUniqChar
============================== 1 failed in 0.01s ===============================

******* 388_Longest_Absolute_File_Path ***********

FAILED 388_Longest_Absolute_File_Path/test_388_Longest_Absolute_File_Path.py::test_lengthLongestPath_empty_input
FAILED 388_Longest_Absolute_File_Path/test_388_Longest_Absolute_File_Path.py::test_lengthLongestPath_no_files
FAILED 388_Longest_Absolute_File_Path/test_388_Longest_Absolute_File_Path.py::test_lengthLongestPath_single_file
FAILED 388_Longest_Absolute_File_Path/test_388_Longest_Absolute_File_Path.py::test_lengthLongestPath_multiple_files
============================== 4 failed in 0.02s ===============================

******* 389_Find_the_Difference ***********

============================== 1 passed in 0.00s ===============================

******* 392_Is_Subsequence ***********


******* 400_Nth_Digit ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked
    from solution import Solution

******* 401_Binary_Watch ***********

FAILED 401_Binary_Watch/test_401_Binary_Watch.py::test_readBinaryWatch - Name...
============================== 1 failed in 0.01s ===============================

******* 404_Sum_of_Left_Leaves ***********

FAILED 404_Sum_of_Left_Leaves/test_404_Sum_of_Left_Leaves.py::test_sumOfLeftLeaves_empty
FAILED 404_Sum_of_Left_Leaves/test_404_Sum_of_Left_Leaves.py::test_sumOfLeftLeaves_single_node
FAILED 404_Sum_of_Left_Leaves/test_404_Sum_of_Left_Leaves.py::test_sumOfLeftLeaves_only_right_nodes
FAILED 404_Sum_of_Left_Leaves/test_404_Sum_of_Left_Leaves.py::test_sumOfLeftLeaves_only_left_nodes
FAILED 404_Sum_of_Left_Leaves/test_404_Sum_of_Left_Leaves.py::test_sumOfLeftLeaves_both_sides
============================== 5 failed in 0.02s ===============================

******* 405_Convert_a_Number_to_Hexadecimal ***********

============================== 3 passed in 0.01s ===============================

******* 408_Valid_Word_Abbreviation ***********

FAILED 408_Valid_Word_Abbreviation/test_408_Valid_Word_Abbreviation.py::test_validWordAbbreviation
============================== 1 failed in 0.01s ===============================

******* 409_Longest_Palindrome ***********

FAILED 409_Longest_Palindrome/test_409_Longest_Palindrome.py::test_longest_palindrome
============================== 1 failed in 0.01s ===============================

******* 412_Fizz_Buzz ***********

============================== 1 passed in 0.00s ===============================

******* 414_Third_Maximum_Number ***********

FAILED 414_Third_Maximum_Number/test_414_Third_Maximum_Number.py::TestSolution::test_thirdMax
============================== 1 failed in 0.02s ===============================

******* 415_Add_Strings ***********

FAILED 415_Add_Strings/test_415_Add_Strings.py::test_addStrings - NameError: ...
============================== 1 failed in 0.01s ===============================

******* 416_Partition_Equal_Subset_Sum ***********

FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_true_for_valid_input
FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_false_for_invalid_input
FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_false_for_empty_input
FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_false_for_None_input
FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_true_for_single_element_input
FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_false_for_odd_sum_input
FAILED 416_Partition_Equal_Subset_Sum/test_416_Partition_Equal_Subset_Sum.py::TestSolution::test_canPartition_returns_true_for_large_input
============================== 7 failed in 0.07s ===============================

******* 421_Maximum_XOR_of_Two_Numbers_in_an_Array ***********

============================== 5 passed in 0.01s ===============================

******* 422_Valid_Word_Square ***********

============================== 5 passed in 0.01s ===============================

******* 434_Number_of_Segments_in_a_String ***********

/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/home/azureuser/verifai/lib/python3.10/site-packages/_pytest/pathlib.py:564: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
<frozen importlib._bootstrap>:1027: in _find_and_load
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
<frozen importlib._bootstrap>:688: in _load_unlocked
    from solution import Solution

******* 437_Path_Sum_III ***********

FAILED 437_Path_Sum_III/test_437_Path_Sum_III.py::TestPathSum::test_empty_tree
FAILED 437_Path_Sum_III/test_437_Path_Sum_III.py::TestPathSum::test_single_node_tree
FAILED 437_Path_Sum_III/test_437_Path_Sum_III.py::TestPathSum::test_binary_tree
============================== 3 failed in 0.01s ===============================

******* 438_Find_All_Anagrams_in_a_String ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/438_Find_All_Anagrams_in_a_String/test_438_Find_All_Anagrams_in_a_String.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from .solution import Solution
E   ImportError: attempted relative import with no known parent package

******* 441_Arranging_Coins ***********

============================== 1 passed in 0.00s ===============================

******* 443_String_Compression ***********

========================= 1 failed, 1 passed in 0.01s ==========================
FAILED 443_String_Compression/test_443_String_Compression.py::test_compress
========================= 1 failed, 1 passed in 0.01s ==========================

******* 448_Find_All_Numbers_Disappeared_in_an_Array ***********

========================= 8 failed, 12 passed in 0.04s =========================
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_9
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_10
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_11
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_12
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_13
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_14
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_16
FAILED 448_Find_All_Numbers_Disappeared_in_an_Array/test_448_Find_All_Numbers_Disappeared_in_an_Array.py::test_findDisappearedNumbers_17
========================= 8 failed, 12 passed in 0.04s =========================

******* 453_Minimum_Moves_to_Equal_Array_Elements ***********

========================= 1 failed, 5 passed in 0.02s ==========================
FAILED 453_Minimum_Moves_to_Equal_Array_Elements/test_453_Minimum_Moves_to_Equal_Array_Elements.py::test_minMoves_when_list_has_both_positive_and_negative_integers_then_returns_correct_number_of_moves
========================= 1 failed, 5 passed in 0.02s ==========================

******* 457_Circular_Array_Loop ***********

    from solution import Solution

******* 458_Poor_Pigs ***********


******* 461_Hamming_Distance ***********

========================= 5 failed, 6 passed in 0.03s ==========================
FAILED 461_Hamming_Distance/test_461_Hamming_Distance.py::TestSolution::test_hammingDistance_large_numbers
FAILED 461_Hamming_Distance/test_461_Hamming_Distance.py::TestSolution::test_hammingDistance_negative_numbers
FAILED 461_Hamming_Distance/test_461_Hamming_Distance.py::TestSolution::test_hammingDistance_one_negative_number
FAILED 461_Hamming_Distance/test_461_Hamming_Distance.py::TestSolution::test_hammingDistance_large_integers
FAILED 461_Hamming_Distance/test_461_Hamming_Distance.py::TestSolution::test_hammingDistance_mixed_integers
========================= 5 failed, 6 passed in 0.03s ==========================

******* 463_Island_Perimeter ***********

========================= 3 failed, 1 passed in 0.02s ==========================
FAILED 463_Island_Perimeter/test_463_Island_Perimeter.py::test_islandPerimeter_one_island
FAILED 463_Island_Perimeter/test_463_Island_Perimeter.py::test_islandPerimeter_multiple_islands
FAILED 463_Island_Perimeter/test_463_Island_Perimeter.py::test_islandPerimeter_invalid_input
========================= 3 failed, 1 passed in 0.02s ==========================

******* 475_Heaters ***********

========================= 3 failed, 7 passed in 0.02s ==========================
FAILED 475_Heaters/test_475_Heaters.py::test_case_3 - assert 2 == 1
FAILED 475_Heaters/test_475_Heaters.py::test_case_4 - assert 2 == 3
FAILED 475_Heaters/test_475_Heaters.py::test_case_9 - assert inf == 0
========================= 3 failed, 7 passed in 0.02s ==========================

******* 479_Largest_Palindrome_Product ***********

FAILED 479_Largest_Palindrome_Product/test_479_Largest_Palindrome_Product.py::test_largestPalindrome
============================== 1 failed in 0.01s ===============================

******* 482_License_Key_Formatting ***********

FAILED 482_License_Key_Formatting/test_482_License_Key_Formatting.py::test_licenseKeyFormatting
============================== 1 failed in 0.01s ===============================

******* 485_Max_Consecutive_Ones ***********

============================== 2 passed in 0.01s ===============================

******* 509_Fibonacci_Number ***********

FAILED 509_Fibonacci_Number/test_509_Fibonacci_Number.py::test_fib - NameErro...
FAILED 509_Fibonacci_Number/test_509_Fibonacci_Number.py::test_fib_raises_exception
============================== 2 failed in 0.01s ===============================

******* 523_Continuous_Subarray_Sum ***********

FAILED 523_Continuous_Subarray_Sum/test_523_Continuous_Subarray_Sum.py::test_checkSubarraySum
============================== 1 failed in 0.02s ===============================

******* 538_Convert_BST_to_Greater_Tree ***********

========================= 1 failed, 1 passed in 0.01s ==========================
FAILED 538_Convert_BST_to_Greater_Tree/test_538_Convert_BST_to_Greater_Tree.py::test_convert_bst_should_return_the_root_with_correct_values
========================= 1 failed, 1 passed in 0.01s ==========================

******* 541_Reverse_String_II ***********


******* 543_Diameter_of_Binary_Tree ***********

========================= 1 failed, 6 passed in 0.02s ==========================
FAILED 543_Diameter_of_Binary_Tree/test_543_Diameter_of_Binary_Tree.py::test_diameter_large_tree
========================= 1 failed, 6 passed in 0.02s ==========================

******* 547_Friend_Circles ***********

========================= 1 failed, 1 passed in 0.02s ==========================
FAILED 547_Friend_Circles/test_547_Friend_Circles.py::test_dfs - assert [1, 1...
========================= 1 failed, 1 passed in 0.02s ==========================

******* 557_Reverse_Words_in_a_String_III ***********

FAILED 557_Reverse_Words_in_a_String_III/test_557_Reverse_Words_in_a_String_III.py::test_solution
============================== 1 failed in 0.01s ===============================

******* 560_Subarray_Sum_Equals_K ***********

============================== 1 passed in 0.00s ===============================

******* 572_Subtree_of_Another_Tree ***********

============================== 1 passed in 0.00s ===============================

******* 581_Shortest_Unsorted_Continuous_Subarray ***********


******* 605_Can_Place_Flowers ***********

FAILED 605_Can_Place_Flowers/test_605_Can_Place_Flowers.py::test_Solution_canPlaceFlowers
============================== 1 failed in 0.01s ===============================

******* 617_Merge_Two_Binary_Trees ***********

FAILED 617_Merge_Two_Binary_Trees/test_617_Merge_Two_Binary_Trees.py::TestSolution::test_mergeTrees_empty
FAILED 617_Merge_Two_Binary_Trees/test_617_Merge_Two_Binary_Trees.py::TestSolution::test_mergeTrees_oneTree
FAILED 617_Merge_Two_Binary_Trees/test_617_Merge_Two_Binary_Trees.py::TestSolution::test_mergeTrees_twoTrees
FAILED 617_Merge_Two_Binary_Trees/test_617_Merge_Two_Binary_Trees.py::TestSolution::test_mergeTrees_differentTreeSizes
FAILED 617_Merge_Two_Binary_Trees/test_617_Merge_Two_Binary_Trees.py::TestSolution::test_mergeTrees_sameTree
============================== 5 failed in 0.02s ===============================

******* 628_Maximum_Product_of_Three_Numbers ***********

FAILED 628_Maximum_Product_of_Three_Numbers/test_628_Maximum_Product_of_Three_Numbers.py::test_maximum_product
============================== 1 failed in 0.01s ===============================

******* 654_Maximum_Binary_Tree ***********

FAILED 654_Maximum_Binary_Tree/test_654_Maximum_Binary_Tree.py::TestSolution::test_constructMaximumBinaryTree_emptyList
FAILED 654_Maximum_Binary_Tree/test_654_Maximum_Binary_Tree.py::TestSolution::test_constructMaximumBinaryTree_singleElement
FAILED 654_Maximum_Binary_Tree/test_654_Maximum_Binary_Tree.py::TestSolution::test_constructMaximumBinaryTree_multipleElements
============================== 3 failed in 0.02s ===============================

******* 665_Non-decreasing_Array ***********

============================== 1 passed in 0.00s ===============================

******* 668_Kth_Smallest_Number_in_Multiplication_Table ***********

FAILED 668_Kth_Smallest_Number_in_Multiplication_Table/test_668_Kth_Smallest_Number_in_Multiplication_Table.py::TestSolution::test_findKthNumber
============================== 1 failed in 0.01s ===============================

******* 671_Second_Minimum_Node_In_a_Binary_Tree ***********

============================== 2 passed in 0.00s ===============================

******* 674_Longest_Continuous_Increasing_Subsequence ***********

========================= 1 failed, 4 passed in 0.02s ==========================
FAILED 674_Longest_Continuous_Increasing_Subsequence/test_674_Longest_Continuous_Increasing_Subsequence.py::TestSolution::test_findLengthOfLCIS_list_with_alternating_values
========================= 1 failed, 4 passed in 0.02s ==========================

******* 680_Valid_Palindrome_II ***********

FAILED 680_Valid_Palindrome_II/test_680_Valid_Palindrome_II.py::test_validPalindrome
============================== 1 failed in 0.01s ===============================

******* 692_Top_K_Frequent_Words ***********

FAILED 692_Top_K_Frequent_Words/test_692_Top_K_Frequent_Words.py::test_topKFrequent_1
FAILED 692_Top_K_Frequent_Words/test_692_Top_K_Frequent_Words.py::test_topKFrequent_2
FAILED 692_Top_K_Frequent_Words/test_692_Top_K_Frequent_Words.py::test_topKFrequent_3
FAILED 692_Top_K_Frequent_Words/test_692_Top_K_Frequent_Words.py::test_topKFrequent_4
FAILED 692_Top_K_Frequent_Words/test_692_Top_K_Frequent_Words.py::test_topKFrequent_5
============================== 5 failed in 0.02s ===============================

******* 695_Max_Area_of_Island ***********

FAILED 695_Max_Area_of_Island/test_695_Max_Area_of_Island.py::test_maxAreaOfIsland
============================== 1 failed in 0.01s ===============================

******* 697_Degree_of_an_Array ***********

FAILED 697_Degree_of_an_Array/test_697_Degree_of_an_Array.py::test_findShortestSubArray
FAILED 697_Degree_of_an_Array/test_697_Degree_of_an_Array.py::test_smallestSubArray
============================== 2 failed in 0.01s ===============================

******* 700_Search_in_a_Binary_Search_Tree ***********

============================== 3 passed in 0.01s ===============================

******* 703_Kth_Largest_Element_in_a_Stream ***********

FAILED 703_Kth_Largest_Element_in_a_Stream/test_703_Kth_Largest_Element_in_a_Stream.py::TestKthLargest::test_kth_largest
FAILED 703_Kth_Largest_Element_in_a_Stream/test_703_Kth_Largest_Element_in_a_Stream.py::TestKthLargest::test_empty_kth_largest
FAILED 703_Kth_Largest_Element_in_a_Stream/test_703_Kth_Largest_Element_in_a_Stream.py::TestKthLargest::test_add_to_heap
FAILED 703_Kth_Largest_Element_in_a_Stream/test_703_Kth_Largest_Element_in_a_Stream.py::TestKthLargest::test_add_small_value
FAILED 703_Kth_Largest_Element_in_a_Stream/test_703_Kth_Largest_Element_in_a_Stream.py::TestKthLargest::test_add_same_value_twice
FAILED 703_Kth_Largest_Element_in_a_Stream/test_703_Kth_Largest_Element_in_a_Stream.py::TestKthLargest::test_add_large_value
============================== 6 failed in 0.03s ===============================

******* 706_Design_HashMap ***********

========================= 3 failed, 1 passed in 0.02s ==========================
FAILED 706_Design_HashMap/test_706_Design_HashMap.py::test_put_get_remove - A...
FAILED 706_Design_HashMap/test_706_Design_HashMap.py::test_duplicate - Attrib...
FAILED 706_Design_HashMap/test_706_Design_HashMap.py::test_remove_nonexistant
========================= 3 failed, 1 passed in 0.02s ==========================

******* 709_To_Lower_Case ***********

============================== 4 passed in 0.01s ===============================

******* 716_Max_Stack ***********

FAILED 716_Max_Stack/test_716_Max_Stack.py::test_push - NameError: name 'MaxS...
FAILED 716_Max_Stack/test_716_Max_Stack.py::test_pop - NameError: name 'MaxSt...
FAILED 716_Max_Stack/test_716_Max_Stack.py::test_top - NameError: name 'MaxSt...
FAILED 716_Max_Stack/test_716_Max_Stack.py::test_peekMax - NameError: name 'M...
FAILED 716_Max_Stack/test_716_Max_Stack.py::test_popMax - NameError: name 'Ma...
============================== 5 failed in 0.02s ===============================

******* 717_1-bit_and_2-bit_Characters ***********


******* 720_Longest_Word_in_Dictionary ***********


******* 724_Find_Pivot_Index ***********

============================== 1 passed in 0.00s ===============================

******* 728_Self_Dividing_Numbers ***********

    from solution import Solution

******* 732_My_Calendar_III ***********

E     Failed: DID NOT RAISE <class 'TypeError'>
732_My_Calendar_III/test_732_My_Calendar_III.py:16: Failed
FAILED 732_My_Calendar_III/test_732_My_Calendar_III.py::test_book - assert No...
FAILED 732_My_Calendar_III/test_732_My_Calendar_III.py::test_book_invalid - F...
FAILED 732_My_Calendar_III/test_732_My_Calendar_III.py::test_duplicate_events
============================== 3 failed in 0.02s ===============================

******* 733_Flood_Fill ***********

FAILED 733_Flood_Fill/test_733_Flood_Fill.py::test_floodFill_dfs - NameError:...
FAILED 733_Flood_Fill/test_733_Flood_Fill.py::test_floodFill_dfs_same_color
FAILED 733_Flood_Fill/test_733_Flood_Fill.py::test_floodFill_bfs - NameError:...
============================== 3 failed in 0.01s ===============================

******* 743_Network_Delay_Time ***********

FAILED 743_Network_Delay_Time/test_743_Network_Delay_Time.py::TestSolution::test_network_delay_time_dfs
FAILED 743_Network_Delay_Time/test_743_Network_Delay_Time.py::TestSolution::test_network_delay_time_dijkstra
FAILED 743_Network_Delay_Time/test_743_Network_Delay_Time.py::TestSolution::test_network_delay_time_invalid_input
FAILED 743_Network_Delay_Time/test_743_Network_Delay_Time.py::TestSolution::test_network_delay_time_large_input
============================== 4 failed in 0.02s ===============================

******* 751_IP_to_CIDR ***********

========================= 1 failed, 2 passed in 0.01s ==========================
FAILED 751_IP_to_CIDR/test_751_IP_to_CIDR.py::test_ipToCIDR - ValueError: neg...
========================= 1 failed, 2 passed in 0.01s ==========================

******* 760_Find_Anagram_Mappings ***********

FAILED 760_Find_Anagram_Mappings/test_760_Find_Anagram_Mappings.py::TestSolution::test_anagramMappings
============================== 1 failed in 0.01s ===============================

******* 766_Toeplitz_Matrix ***********

FAILED 766_Toeplitz_Matrix/test_766_Toeplitz_Matrix.py::test_toeplitz_matrix
============================== 1 failed in 0.01s ===============================

******* 771_Jewels_and_Stones ***********

FAILED 771_Jewels_and_Stones/test_771_Jewels_and_Stones.py::TestSolution::test_numJewelsInStones
============================== 1 failed in 0.01s ===============================

******* 784_Letter_Case_Permutation ***********

============================== 1 passed in 0.00s ===============================

******* 804_Unique_Morse_Code_Words ***********

FAILED 804_Unique_Morse_Code_Words/test_804_Unique_Morse_Code_Words.py::test_solution
============================== 1 failed in 0.01s ===============================

******* 811_Subdomain_Visit_Count ***********

FAILED 811_Subdomain_Visit_Count/test_811_Subdomain_Visit_Count.py::test_subdomainVisits
============================== 1 failed in 0.01s ===============================

******* 819_Most_Common_Word ***********

FAILED 819_Most_Common_Word/test_819_Most_Common_Word.py::test_mostCommonWord
============================== 1 failed in 0.01s ===============================

******* 832_Flipping_an_Image ***********

========================= 9 failed, 1 passed in 0.04s ==========================
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenSingleEmptyList_returnsSingleEmptyList
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenSingleListWithASingle0_returnsSingleListWithASingle1
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenSingleListWithASingle1_returnsSingleListWithASingle0
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenSingleListWithMultiple0s_returnsSingleListWithMultiple1s
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenSingleListWithMultiple1s_returnsSingleListWithMultiple0s
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenMultipleSingleLists_returnsMultipleSingleLists_InvertedAndReversed
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenMultipleLists_returnsMultipleLists_InvertedAndReversed
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenALargeList_returnsLargeList_InvertedAndReversed
FAILED 832_Flipping_an_Image/test_832_Flipping_an_Image.py::test_flipAndInvertImage_whenGivenALargeList_returnsLargeList_InvertedAndReversed_AlternativeMethod
========================= 9 failed, 1 passed in 0.04s ==========================

******* 836_Rectangle_Overlap ***********

========================= 1 failed, 5 passed in 0.02s ==========================
FAILED 836_Rectangle_Overlap/test_836_Rectangle_Overlap.py::test_intersect_helper_function
========================= 1 failed, 5 passed in 0.02s ==========================

******* 844_Backspace_String_Compare ***********

FAILED 844_Backspace_String_Compare/test_844_Backspace_String_Compare.py::test_backspaceCompare
============================== 1 failed in 0.02s ===============================

******* 852_Peak_Index_in_a_Mountain_Array ***********

FAILED 852_Peak_Index_in_a_Mountain_Array/test_852_Peak_Index_in_a_Mountain_Array.py::test_peakIndexInMountainArray
============================== 1 failed in 0.01s ===============================

******* 867_Transpose_Matrix ***********

FAILED 867_Transpose_Matrix/test_867_Transpose_Matrix.py::test_transpose - In...
============================== 1 failed in 0.02s ===============================

******* 868_Binary_Gap ***********


******* 872_Leaf-Similar_Trees ***********

FAILED 872_Leaf-Similar_Trees/test_872_Leaf-Similar_Trees.py::test_leaf_similar
============================== 1 failed in 0.02s ===============================

******* 876_Middle_of_the_Linked_List ***********

ImportError while importing test module '/data/sandeep/leetcode-master/.verifaiws/876_Middle_of_the_Linked_List/test_876_Middle_of_the_Linked_List.py'.
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
    from solution import ListNode, Solution
E   ImportError: cannot import name 'ListNode' from 'solution' (/data/sandeep/leetcode-master/.verifaiws/876_Middle_of_the_Linked_List/solution.py)

******* 904_Fruit_Into_Baskets ***********

FAILED 904_Fruit_Into_Baskets/test_904_Fruit_Into_Baskets.py::test_totalFruit1
FAILED 904_Fruit_Into_Baskets/test_904_Fruit_Into_Baskets.py::test_totalFruit2
FAILED 904_Fruit_Into_Baskets/test_904_Fruit_Into_Baskets.py::test_totalFruit3
FAILED 904_Fruit_Into_Baskets/test_904_Fruit_Into_Baskets.py::test_totalFruit4
FAILED 904_Fruit_Into_Baskets/test_904_Fruit_Into_Baskets.py::test_totalFruit5
============================== 5 failed in 0.02s ===============================

******* 905_Sort_Array_By_Parity ***********

========================= 1 failed, 4 passed in 0.02s ==========================
FAILED 905_Sort_Array_By_Parity/test_905_Sort_Array_By_Parity.py::test_sortArrayByParity_oddNumbersFirst
========================= 1 failed, 4 passed in 0.02s ==========================

******* 922_Sort_Array_By_Parity_II ***********

FAILED 922_Sort_Array_By_Parity_II/test_922_Sort_Array_By_Parity_II.py::test_sortArrayByParityII
============================== 1 failed in 0.01s ===============================

******* 929_Unique_Email_Addresses ***********

========================= 3 failed, 2 passed in 0.02s ==========================
FAILED 929_Unique_Email_Addresses/test_929_Unique_Email_Addresses.py::test_numUniqueEmails_only_duplicates
FAILED 929_Unique_Email_Addresses/test_929_Unique_Email_Addresses.py::test_numUniqueEmails_no_duplicates
FAILED 929_Unique_Email_Addresses/test_929_Unique_Email_Addresses.py::test_numUniqueEmails_invalid_emails
========================= 3 failed, 2 passed in 0.02s ==========================

******* 933_Number_of_Recent_Calls ***********

FAILED 933_Number_of_Recent_Calls/test_933_Number_of_Recent_Calls.py::test_recent_counter
============================== 1 failed in 0.01s ===============================

******* 937_Reorder_Log_Files ***********

========================= 3 failed, 2 passed in 0.02s ==========================
FAILED 937_Reorder_Log_Files/test_937_Reorder_Log_Files.py::test_reorderLogFiles_with_only_letter_logs
FAILED 937_Reorder_Log_Files/test_937_Reorder_Log_Files.py::test_reorderLogFiles_with_mixed_logs
FAILED 937_Reorder_Log_Files/test_937_Reorder_Log_Files.py::test_reorderLogFiles_with_letter_logs_with_same_content
========================= 3 failed, 2 passed in 0.02s ==========================

******* 945_Minimum_Increment_to_Make_Array_Unique ***********

============================== 5 passed in 0.01s ===============================

******* 946_Validate_Stack_Sequences ***********

========================= 5 failed, 15 passed in 0.04s =========================
FAILED 946_Validate_Stack_Sequences/test_946_Validate_Stack_Sequences.py::test_validateStackSequences_3
FAILED 946_Validate_Stack_Sequences/test_946_Validate_Stack_Sequences.py::test_validateStackSequences_12
FAILED 946_Validate_Stack_Sequences/test_946_Validate_Stack_Sequences.py::test_validateStackSequences_15
FAILED 946_Validate_Stack_Sequences/test_946_Validate_Stack_Sequences.py::test_validateStackSequences_16
FAILED 946_Validate_Stack_Sequences/test_946_Validate_Stack_Sequences.py::test_validateStackSequences_18
========================= 5 failed, 15 passed in 0.04s =========================

******* 953_Verifying_an_Alien_Dictionary ***********

========================= 1 failed, 6 passed in 0.02s ==========================
FAILED 953_Verifying_an_Alien_Dictionary/test_953_Verifying_an_Alien_Dictionary.py::test_isAlienSorted_empty_order
========================= 1 failed, 6 passed in 0.02s ==========================

******* 954_Array_of_Doubled_Pairs ***********

========================= 1 failed, 3 passed in 0.01s ==========================
FAILED 954_Array_of_Doubled_Pairs/test_954_Array_of_Doubled_Pairs.py::test_case_4
========================= 1 failed, 3 passed in 0.01s ==========================

******* 961_N-Repeated_Element_in_Size_2N_Array ***********


******* 962_Maximum_Width_Ramp ***********

========================= 5 failed, 7 passed in 0.06s ==========================
FAILED 962_Maximum_Width_Ramp/test_962_Maximum_Width_Ramp.py::test_maxWidthRamp_all_values_same
FAILED 962_Maximum_Width_Ramp/test_962_Maximum_Width_Ramp.py::test_maxWidthRamp_duplicate_values
FAILED 962_Maximum_Width_Ramp/test_962_Maximum_Width_Ramp.py::test_maxWidthRamp_negative_values
FAILED 962_Maximum_Width_Ramp/test_962_Maximum_Width_Ramp.py::test_maxWidthRamp_all_negative_values
FAILED 962_Maximum_Width_Ramp/test_962_Maximum_Width_Ramp.py::test_maxWidthRamp_mixed_values
========================= 5 failed, 7 passed in 0.06s ==========================

******* 973_K_Closest_Points_to_Origin ***********


******* 977_Squares_of_a_Sorted_Array ***********


******* 981_Time_Based_Store ***********

============================== 4 passed in 0.01s ===============================

******* 997_Find_The_Town_Judge ***********

========================= 1 failed, 9 passed in 0.02s ==========================
FAILED 997_Find_The_Town_Judge/test_997_Find_The_Town_Judge.py::test_findJudge_10
========================= 1 failed, 9 passed in 0.02s ==========================

******* __init__ ***********


******* logs ***********


